<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Untitled Document</title>
<style>
img {
	width: 65%;
	height: 70%;
	overflow: auto;
    display: block;
    margin-left: auto;
    margin-right: auto;
}
</style>

</head>

<body>
<h2 align="center">CONTENT</h2>

<h2><p align="center"><a href="#_Toc80803537"><strong>Introduction to PANC-DB</strong></a> <br />
  <a href="#_Toc80803540"><strong>Workflow and Protocols</strong></a> <br />
  <a href="#_Toc80803541"><strong>Donor Information page</strong></a> <br />
  <a href="#_Toc80803542"><strong>Experimental Data Download</strong></a> <br />
  <a href="#_Toc80803544"><strong>Interactive Analysis with CellxGene</strong></a><br/>
  <a href="#_Toc80803543"><strong>News and publications page</strong></a></p>
<p align="center"><strong>&nbsp;</strong></p></h2>
<ol type="A">
<h2 style="font-size:1.5vw"><li><a name="_Toc80803537" id="_Toc80803537"><strong>Introduction to  PANC-DB</strong></a></li></h2>

<p style="font-size:1.2vw;">The PANC-DB portal, developed and maintained  by the Data integration and distribution core of HPAP, is the source for all  experimental data and metadata from both the T1D and T2D programs of HPAP  consortium. The PANC-DB hosts raw data generated by participating cores using a wide-range of deep phenotyping assays. Our data repository was recently upgraded to improve data navigation by implementing new visualization and analysis tools, all described in subsequent sections below.<br/> 
<br/>	
The Getting started page is meant to serve as a pictorial guide to aid new users with navigating PANC-DB, and become familiar with various features for data access, download and analysis. Figure 1 below shows PANC-DB's homepage with the four primary data access options and the footnote section with more links to resources. </p>
	<br/>
	<img src="/project/hpap/public/assets/getting_started/images/new-website.png" alt="new landing page" style="vertical-align:middle"/>
<p align="left" style="font-size:1.2vw;">
  <strong>Figure 1: PANC-DB Landing page with four primary data access menu, navigation bar, footnote and news status</strong></p>
<p>&nbsp;</p>
	
<p style="font-size:1.2vw;">The four main icons in figure 1  takes you to different data access and download portals where you can do the following-</p>
<ol type="1" style="font-size:1.3vw">
 <li><p style="font-size:1.3vw;"> <strong>Donor Information</strong>: <a href="https://hpap.pmacs.upenn.edu/explore/donor?by_donor">hpap.pmacs.upenn.edu/explore/donor?by_donor</a><br/>
<br/>
	This option takes users to a poral with snapshot of donor clinical data such as demographic info, medical history, auto-antibody result, HLA data and more. Users can also access the image and clinical data viewers to aid with decision making and cohort preparation. See section C.1 for details.</p></li>
<li><p style="font-size:1.2vw;"><strong>Experimental Data Download</strong>: <a href="https://hpap.pmacs.upenn.edu/explore/download?matrix">hpap.pmacs.upenn.edu/explore/download?matrix</a><br/><br/>
This new assay-centric <em>Data Download Matrix</em> capability was developed to improve data navigation and download. See section C.2 for details. </p></li>
<li><p style="font-size:1.3vw;"><strong>Interactive Analysis</strong>: <a href="https://hpap.pmacs.upenn.edu/analysis">hpap.pmacs.upenn.edu/analysis</a><br/>
	<br/>
	The Faryabi laboratory implemented this new page that provides the first interactive data analysis portal for pancreatic single-cell RNA-seq data. The details of the new Interactive Analysis functionality are presented in Section C.2. </li></p>
<li><p style="font-size:1.3vw;"><strong>Data Status</strong>: <a href="https://hpap.pmacs.upenn.edu/explore/download?dataStatus">hpap.pmacs.upenn.edu/explore/download?dataStatus</a><br/>
	<br/>
	PANC-DB has offered the improved <em>Data Status</em> matrix to communicate the state of data from various assays for each donor. The goal is to indicate whether particular experiment data for a given donor is <em>available</em>, <em>pending</em>, <em>revoked</em>, or <em>never</em> coming. See section C.2 for details. </li>
	</p>
</ol>
	
<p style="font-size:1.3vw;">In this document we are utilizing a use case scenario to walk you through the database and show you how you can use the repository to your own benefit. We will be using <em>Islet Perifusion</em> assay as our example, which is an islet functional assay performed at two participating institutes under HPAP.</p>
<br/>


</p>
<h2 style="font-size:1.5vw"><li><a name="_Toc80803540" id="_Toc80803540"><strong>Workflow and Protocols
	</strong></a></li></h2>
	<p style="font-size:1.3vw;">In this section you can find the SOPs for all assays and anaysis hosted on PANC-DB and download them as a PDF. Each SOP has a workflow tab showing how data is collected, processed and generated. Figures 2 to 4 below show how to access the workfllow and protocol menu in general and view the SOP for Islet Perifusion assay on PANC-DB.</p> 
		<br/>
	<img src="/project/hpap/public/assets/getting_started/images/main_workflow_menu.png" alt="new_workflow" style= "vertical-align:middle"/>
	<p align="left" style="font-size:1.2vw;">
  <strong>Figure 2: Workflow & Protocols drop down menu on the navigation bar</strong></p>
<p>&nbsp;</p>
	
	<img src="/project/hpap/public/assets/getting_started/images/view-perifusion-workflow.png" alt="perifusion_workflow" style= "vertical-align:middle"/>
	<p align="left" style="font-size:1.2vw;">
  <strong>Figure 3: Islet physiology workflow and selecting Islet perifusion protocol from the Islet physiology drop down menu</strong></p>
<p>&nbsp;</p>
	
	<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\view-islet-perifusion.png" alt="perifusion sop" style= "vertical-align:middle"/>
<p align="left" style="font-size:1.2vw;">
  <strong>Figure 4: SOP for Islet perifusion performed at University of Pennsylvania </strong></p>
<p>&nbsp;</p>
	
<h2 style="font-size:1.5vw"><a name="_Toc80803541" id="_Toc80803541"><li><strong>Donor Information Page</strong></li></a></h2>

	
<p style="font-size:1.3vw;">Let&rsquo;s start with the <a href="https://hpap.pmacs.upenn.edu/explore/donor?by_donor">Donor Information</a> page which is a summary view  of our donor datasets. It contains donor  cards summarizing a donor’s key clinical and demographic information. The donor cards also provides a snapshot of endocrine cell percentages for <em>alpha, beta, acinar, PP</em> and <em>epsilon</em> cells as obtained from CyTOF analysis. Figure-5 below explains the donor card display in depth, with details on  how to find more information on a particular donor and navigate to the donor-specific tabs.</p><br/>
	
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\Donor_information_new.png" alt="donor cards" style= "vertical-align:middle"/>
	<p align="left" style="font-size:1.2vw;">
  <strong>Figure 5: New Donor information page showing donor-cards, donor filters and ways to navigate to other tabs </strong></p>
<p>&nbsp;</p>
	
<p style="font-size:1.3vw;">One can use the donor filters on the left side of the page to create a cohort using key donor phenotypes i.e Democraphics such as <em>Age</em>, <em>Gender</em>, and <em>Race</em> and Medical characteristics such as <em>disease diagnosis</em> (T1D, T2D or No diabetes), <em>HbA1c (%)</em> values and more. Figure-6  describes how to use every option on the filter to browse your data or sample of interest.</p>
<br />
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\new-filter-collapsed.png" alt="donor filter" style= "vertical-align:middle"/>
<div>
  <p align="left" style="font-size:1.3vw;"><strong>Figure 6: Donor filter options for all data download pages</strong></p>
</div>
<br />

<p style="font-size:1.3vw;">Once you have your subset of  interest i.e. donors with the selected phenotypes, you can navigate  to the <em>donor specific</em> tabs for further exploration of specific data. See figure 7 for details on what you will  see in this section of the web portal and how to download specific files from that window.</p>
<br />
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\image-viewers.png" alt="donor-specific-tab" style= "vertical-align:middle"/>

<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 7: Donor specific window for HPAP-002</strong></p>
</div>
<br />

<p style="font-size:1.3vw;">The donor specific window for all  donors have the following tabs-</p>
<ol type="a" style="font-size:1.2vw">
  <li><p style="font-size:1.3vw;"><strong>Clinical</strong>-  Detailed medical and clinical history of the donor with the ability to download  them as a <em>.docx</em> and <em>.xlsx</em> file. See <em>figure-7</em> above for details</p></li>
  <li><p style="font-size:1.3vw;"><strong>Islet isolation</strong>- information related to islet isolation and  quality for a particular donor. It also contains images of the donor pancreas  pre and post fat trimming and islets. Figure 8 below shows the Islet isolation tab for HPAP-001</p></li>
	<br/>
	<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\view-isolation-data.png" alt="islet-isolation-tab" style= "vertical-align:middle"/>

<div>
  <p align="center" style="font-size:1.2vw;"><strong>Figure 8:Islet isolation viewer for donor HPAP-001</strong></p>
</div>
<br />
	
  <li><p style="font-size:1.3vw;"><strong>Islet function</strong>- Viewer for the <em>Islet perifusion</em> and <em>Oxygen consumption</em> plots for each donor. Figures 9 and 10 show the viewer on PANC-DB and the link-out option to IIDP portal respectively.</p></li>
  <li><p style="font-size:1.3vw;"><strong>Islet composition</strong>- Visualization to show <em>endocrine cell counts</em> post <em>CyTOF</em> as well as the <em>tSNE </em>plot for the  donor. Figure 11 shows a screenshot of CyTOF pie chart for HPAP-001.</p></li>
	<br/>
  
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\view-iidp-link.png" alt="islet-physiology-tab" style= "vertical-align:middle"/>

<div>
  <p align="center" style="font-size:1.2vw;"><strong>Figure 9: Islet Perifusion ad Oxygen consumption plots along with link to visit IIDP database for HPAP-004</strong></p>
</div>
	<br/>
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\iidp-plot.png" alt="iidp-link" style= "vertical-align:middle"/>

<div>
  <p align="center" style="font-size:1.2vw;"><strong>Figure 10: Interactive Islet perifusion plot on IIDP portal with dono-to-donor link to Pancreatlas and back to PANC-DB</strong></p>
</div>
</p>
<br/>
<li><p style="font-size:1.3vw;"><strong>Imaging mass cytometry</strong>- Image viewer showing selected IMC slides for  a donor. This is a collection of only a subset of the best captured images. To  view the remainder of the images, you can either navigate to <em>Pancreatlas&rsquo;s</em> advanced viewer linked to  this tab or download the raw images and import them to your own image viewing  application. Figure 11 has a screenshot of the IMC viewer on PANC-DB with its standard anatomy and immune panel filters.</p></li>

<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\view-histology-imc.png" alt="imc-histology-viewer" style= "vertical-align:middle"/>
<div>
  <p align="center" style="font-size:1.2vw;"><strong>Figure 11: Histology and IMC image viewers with fiters for HPAP-003 on PANC-DB
	  </strong></p>
</div>
</p><br/>
  <li><p style="font-size:1.3vw;"><strong>Histology imaging</strong>- Similar to the IMC viewer, this tab is an  application to view a subset of the best Histology slides for the donor. Remainder of the images can be viewed on <em>Pancreatlas</em> or the raw <em>svs</em> files can be  downloaded. Figure 11 shows a screenshot of the Histology viewer with its anatomy and magnification filters.</p></li>

  <li><p style="font-size:1.3vw;"><strong>Sequencing data</strong>- this tab is a direct way to access the  sequencing files hosted on PANC-DB for a particular donor. It also provides the option to visit <em>CMDGA</em> portal for the same data. CMDGA provides advanced visualization and navigation features.</p></li>
</ol>
</p>
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\seq-view.png" alt="sequencing-tab" style= "vertical-align:middle"/>
<div>
  <p align="center" style="font-size:1.2vw;"><strong>Figure 12: Sequencing data tab for HPAP-003 </strong></p>
</div>
</p><br/>


  <h2 style="font-size:1.5vw"><a name="_Toc80803542" id="_Toc80803542"><li><strong>Experimental Data Download page </strong></li></a></h2>

<p style="font-size:1.3vw;">Now let&rsquo;s move on to our data download pages with user-friendly matrix view of our data. Users can retrieve their data based on various intuitive and relevant filters and download them in bulk or for single donors. An important upgrade to this feature is the ability to also use the Donor filters to select a subset of donor phenotypes such as disease type, demographics etc to create a subset.</p>

<ol type="1" style="font-size: 1.3vw">
	
<li><strong>Download By Cell type/Tissue-</strong></li>
	<p style="font-size:1.3vw;">This page organizes our data into meaningful experimental units and categorizes data by <em>cell type</em> and <em>tissue</em> on which various assays were performed to genertate the raw data. Hence, Available data are organized along two dimensions i.e., Assay type and Tissue/Cell type, giving a concise overview of available data.<br/><br/>The experimental unit differs for every assay, for instance for IMC, data is collected for three pancreatic region i.e. head, body and tail of pancreas, each region making 1 unit so a total of 3 units per donor. Figure 13 shows available experiments for Islet perifusion assay for all donors on PANC-DB.</p>
	<p align="center">
</p>
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\by-cell-tissue-type.png" alt="by cell/tissue tab" style= "vertical-align:middle"/>
<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 13: Download by Cell/Tissue type matrix showing 108 experiments for Islet Perifusion and ways to download the data in bulk</strong></p>
</div>
<br />
<li><strong>Download By Data status & Donors-</strong></li>
	<p style="font-size:1.3vw;">This page also provides a matrix view of our data categorized by different availability status per donor sample. Each cell in the matrix shows the number of experiments for a given <em>Status-Assay</em> combination per donor. This provides a quick and intuitive overview of all data. The four data availabilty criterias on this page are- <br/>
		<ul type="disc">
			<li><u>Available</u>- data that is currently available on PANC-DB</li>
			<li><u>Pending</u>- this is the default data status i.e. data that is expected to be available  on PANC-DB</li>
			<li><u>Never</u>- data that is never expected and hence will never be available on PANC-DB</li>
			<li><u>Revoked</u>- data that was posted on PANC-DB but subsequently removed since the generating core deemed it unusable</li>
		</ul><br/>
	The figure below shows status for <em>Islet Perifusion</em> data for type-1 diabetes (T1D) donors in the new <em>Data Status Matrix</em>. The four different criteias are color coded. When hovering over a cell, the corresponding row and column headers are highlighted for better navigation. Both direct download as a zip file (size <=5 GB) as well as SFTP (size >5 GB) are available. </p>
		<p align="center"> <br />
</p>
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\by-data-status.png" alt="by data status tab" style= "vertical-align:middle"/>
<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 14: Data status matrix highlighting "available" Islet perifusion data for T1D donors in yellow </strong></p>
</div>
<br />
		
<li><strong>Download By Donor List-</strong></li>
	<p style="font-size:1.3vw;">This page lists all donor datasets live on  PANC-DB by their HPAP IDs and clicking on each donor will reveal the underlying sub-folder organization that  contain downloadable data. Figure 15 lists <em>T1D, T2D and No Diabetes</em> donors and their sub-folders. In order to download any data, navigate  to the desired folder or sub-folder and select appropriate checkboxes. You can  either download single files or the entire collection of assay folders for the selected donor. You can select the <em>View data structure</em> button on top right, to view a schema of folder organization</p><br/>
		
	<br/>
	<p style="font-size:1.3vw;"><strong>The list of donors on this page does not represent the number of donors accepted by the HPAP program.</strong> It usually takes upto 14 days to make a recently accepted donor live on PANC-DB. The list however represents the  donors currently live (visible) on the database with at least one assay  data available for download. The basic  requirement for a donor to be live on PANC-DB is, to have the clinical data ready to display and download. This activates the <em>donor cards</em> on the Donor information page.</p>

<p align="center">
</p>
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\view-list.png" alt="donor list view" style= "vertical-align:middle"/>
<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 15: Donor list view with the metadata download option</strong></p>
</div>
<br />
<p style="font-size:1.3vw;">When trying to download data, you might be prompted to register yourself or login as shown in figure 16. It&rsquo;s a simple three step process explained <a href="https://hpap.pmacs.upenn.edu/explore/register">here</a>. <strong>Refer  to figure 16 for ways to avoid  downloading empty folders from the list view. Users can  download assay metadata on this page as well</strong>. When you download the metadata, you  will acquire a zip file with a stack of spreadsheets for each assay, as shown in figure 17. You will also download a &ldquo;README&rdquo;  file with information on each assay, as well as the file &ldquo;PancDB_Donors&rdquo; with  donor demographic information, RRIDs, and relevant links. </strong></p>
<br /><br />
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\hide-prompt.png" alt="hide-folder" style= "vertical-align:middle"/>

<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 16: a) Users must register/login to download any data b) How to hide empty directories before download </strong></p>
</div>
<br />
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\metadata-file-stack.png" alt="metadata-stack" style= "vertical-align:middle"/>

<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 17: Collection of metadata spreadseet all downloaded as a zip file from PANC-DB</strong></p>
</div>

<strong><br clear="all" />
</strong>

</p>
</ol>
</p>
<h2 style="font-size:1.5vw"><li><a name="_Toc80803543" id="_Toc80803543"><strong>News and publications page</strong></a></li></h2>
<p style="font-size:1.2vw;">The News and Publications section is divided into two parts-</p>
<ol type="1" style="font-size:1.3vw">
  <li><p style="font-size:1.3vw;"><strong>Data  release page</strong>- this section contains updates  related to HPAP data and PANC-DB features. As you can see in figure 18 below, any changes to data  hosted on PANC-DB will be posted here. <strong>If  any data is revoked for any reason and replaced with newer versions, links to those legacy files will also be available</strong> in  the <a href="https://hpap.pmacs.upenn.edu/explore/news?datarelease"><em>data release</em></a> page. </p></li>
  <li><p style="font-size:1.3vw;"><strong>Publications page</strong>-In this section all HPAP related  publications are listed in a chronological order. Direct links to the online copy of the  articles is provided for each listed study in the <a href="https://hpap.pmacs.upenn.edu/explore/news?publications">Publications</a> tab so users can download the full text articles. </p></li>
</ol>
<br />
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\data-release-pubs.png" alt="news-pub-page" style= "vertical-align:middle"/>

<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 18: News and Publications page on PANC-DB showing a) latest version of metadata and links to legacy data b) latest publications using HPAP data</strong></p>
	<br/>
</div>
<h2 style="font-size:1.5vw"><li><a name="_Toc80803544" id="_Toc80803544"><strong>Interactive analysis with CellxGene</strong></a></li></h2>

<p style="font-size:1.3vw;">We have now expanded the role of PANC-DB as a data analysis platform by introducing the first interactive data analysis portal for pancreatic single-cell RNA-seq data. 

To provide this one-of-a-kind capability, the team has collected, harmonized, and processed transcriptomic data of more than 270,000 cells from more than 70 donors. The result of this massive single-cell transcriptomic analysis is provided as a <strong>Seurat object</strong> to the PANC-DB users. Faryabi lab has developed a code that allows conversion of the Seurat object to <strong>AnnData binary object</strong> that is required for inputting data to PANC-DB's CellXGene (figure 19). </p><br/>

<br />
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\Interactive-analysis.png" alt="interactive-analysis-page" style= "vertical-align:middle"/>

<div>
  <p align="center" style="font-size:1.2vw;"><strong>Figure 19: PANC-DB's Interactive Anlaysis Page</strong></p>
	<br/>
</div>

<p style="font-size:1.3vw;">
<br/>PANC-DB's CellxGene offers a number of unique features and can be used for a variety of interactive data exploration. We have demonstrated some of the functionalities with a use case scenario: 

<ol type: "1" style="font-size:1.3vw">

<li>Viewing multiple embeddings of the same dataset, like UMAP, T-SNE, and PCA. For instance, a user can color the UMAP presentation of cells by their predicted cell types as shown in figure 20 and figure 21.</li><br/>
	
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\cg_default.png" alt="cellxgene-landing-page" style= "vertical-align:middle"/>

<div>
  <p align="center" style="font-size:1.2vw;"><strong>Figure 20: Landing page for PANC-DB's CellxGene analysis</strong></p>
	<br/>
</div>
	
	<br/><br/>	
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\cg-colored.png" alt="cellxgene-colored-page" style= "vertical-align:middle"/>

<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 21: Cell annotations colored on UMAP. On the left menu bar, the color map by “cell_type” is chosen, which highlights the cells by its classification</strong></p>
	<br/>
</div>
	
<li>Sub-setting cells by cell annotations, which can be enumerated or continuous values (Figure 20).</li><br/>
<li>Users can also select any subset of cells by drawing a closed curve encompassing the cells (figure 21). This option allows users to mark different subsets of cells for downstream analysis.</li>
	
<br/><br/>	
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\cg-lasso.png" alt="cellxgene-lasso-tool" style= "vertical-align:middle"/>

<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 22: Demonstration of the lasso tool which was used to select a circle of cells on the UMAP</strong></p>
	<br/>
</div>
	
<li> User can visualize expression level of a particular gene across various populations by coloring the UMAP. For instance, the examination of INS expression shows elevated level of this gene in <em>Beta</em> cells, as expected (figure 22).</li>
<br/>	
<br/>	
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\cg-INS.png" alt="cellxgene-INS-expression" style= "vertical-align:middle"/>

<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 22: UMAP is colored by INS expression, which shows the elevated expression of INS (darker green to blue) in the annotated Beta cells located on the UMAP lower right part. Histograms on the left side menu show the INS gene count distribution on various annotated cell types</strong></p>
	<br/>
</div>	
<li>One can compare the expression of two genes across cells from different donors on a bivariate plot. For instance, comparing expression of INS and GCG shows that their expression levels are not correlated independent of the disease condition (figure 23). One can compare any number of genes by adding them t the gene list on the right side panel as shown in figure 23.</li>
<br/>	
<li>CellxGene also offers various ways to interactively compare gene expression between two groups of cells. For instance, differential gene expression analysis identifies GCG as the most upregulated gene in Alpha compared to Beta cells (figure 24). Currently, the differential expression functionality has an upper limit of 50,000 cells (between population 1 and population 2). Hence, users need to be strategic about targetting types of differential expression tests to run on large datasets. </li>
</p>	
<br/>	
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\cg-GCG.png" alt="cellxgene-INS-GCG-expression" style= "vertical-align:middle"/>

<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 23: Scatterplot comparing INS and GCG expressions in cells from <em>T1D, T2D, control</em> and <em>AAB</em> donors, marked by <em>“disease_state”</em> color from the left side menu. As expected, INS and GCG expressions are not correlated.</strong></p>
	<br/>	
	</div>
	
<br/>	
<img src="C:\Users\nsamanta\Box\HPAP-Curators\Website content\Help page\help-page-pics\New website\annotated images\cg-diff-expression.png" alt="cellxgene-differential-expression" style= "vertical-align:middle"/>

<div>
  <p align="left" style="font-size:1.2vw;"><strong>Figure 24: Results of differential expression of <em>“Alpha”</em> vs <em>“Beta”</em> cell types. The resulting gene list, shown on the right-hand side, identifies GCG as the most upregulated gene in the Alpha compared to Beta cells.</strong></p>
	<br/>	
	</div>

</ol>
</ol>
</ol>

</body>
</html>
