<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Untitled Document</title>
<style>
img {
    display: block;
    margin-left: auto;
    margin-right: auto;
}
</style>
</head>

<body>
<img src="/assets/workflow/Islet Physiology Studies/HPAP-logo.jpg" alt="perifusion" width="200" height="90" style= "vertical-align:middle"/>
<h3 align="center"><strong>ATAC  Sequencing Protocol</strong></h3>
<p align="center">Kaestner Lab<br />
  Date revised: February 20,  2020</p>
<ol type="I">
  <li><strong>Reagents</strong></li><br/>
<ol type="1">
  <li>1X PBS</li>
  <li>Nuclease-free  H2O</li>
  <li>NP-40  10% (Sigma/Roche, catalog # 11332473001), store at 4oC</li>
  <li>Tween-20  10% (Sigma/Roche, catalog # 11332465001), store at 4oC</li>
  <li>Digitonin  (Promega, catalog # G9441) - Dilute 1:1 with water to make 1% working stock,  aliquot, store at -20oC for up to 6 months, do not freeze/thaw more  than 5 times</li>
  <li>1M  Tris-HCl, pH 7.5 (ThermoFisher, catalog # 15567-027)</li>
  <li>5M NaCl</li>
  <li>1M MgCl2  (Sigma, catalog # M1028)</li>
  <li>Tagment  DNA Enzyme 1 (TDE1) (Illumina, catalog # 15027865)</li>
  <li>Tagment DNA Buffer (Illumina,  catalog # 15027866)</li>
  <li>MinElute Reaction Cleanup Kit  (Qiagen, catalog # 28204)</li>
  <li>Primers (see table at end of protocol)</li>
  <li>NEBNext High-Fidelity  2X PCR Master Mix (NEB, catalog # M0541S)</li>
  <li>SYBR Green I  (ThermoFisher, catalog # S7563) </li>
  <li>Agencourt AMPure XP magnetic  beads (Beckman Coulter, catalog # A63880)</li>
  <li>80% Ethanol (made fresh)</li>
  <li>Agilent High Sensitivity DNA  Bioanalysis Kit (Agilent, catalog # 5067-4626)</li>
  <li>Qubit dsDNA HS Assay Kit  (ThermoFisher, catalog # Q32851)</li>
  <li>Qubit fluorometer (ThermoFisher)</li>
  <li>Agilent Bioanalyzer</li>
</ol><br/>
<ul type="disc">
<p><strong><em>Notes: This protocol is based on the Buenrostro  et al. papers </em></strong><strong><em>[1, 2]</em></strong><strong><em>, with the following  modifications:<u></u></em></strong></p>

  <li><strong><em>Adjusted lysis step and transposition buffer based on Omni-ATAC  protocol </em></strong><strong><em>[3]</em></strong><strong><em>, which in our hands yielded  larger number of peaks (~2x) and reduced mitochondrial reads by ~20%.</em></strong></li>
  <li><strong><em>We found that column purification of the libraries does not remove  primer-dimers (78 bp), and our libraries often contained large fragments  1,000-10,000 bp in length.&nbsp; Therefore, we  use magnetic bead purification rather than column purification of the  libraries.&nbsp; We recommend that you perform  bioanalysis of your initial libraries to determine whether double-sided bead  purification is needed.</em></strong></li>
  <li><strong><em>Typically, 50,000-100,000 cells yield the best results.&nbsp; As few as 5,000 have been reported to  work.&nbsp; The ratio of transposase to cell  number is important.&nbsp; We recommend that,  if possible, you test different cell numbers and different transposase cell  ratios.&nbsp; The Buenrostro et al. papers are  based on using 50,000 cells.&nbsp; If we use  100,000 cells, then we double the volume of all reagents in the protocol.</em></strong></li>
  <li><strong><em>We use live, freshly-isolated cells, not frozen.&nbsp; Reportedly, this revised protocol (Omni-ATAC)  can be performed on snap-frozen cells or tissue </em></strong><strong><em>[3]</em></strong><strong><em>.&nbsp; Alternatively, you can slowly freeze cells  using the protocol detailed in </em></strong><strong><em>[4]</em></strong><strong><em> (resuspend pellet in  CryoStor buffer 50 ul / 50,000 cells and freeze at -80oC in Mr.  Frosty device).&nbsp; Pipet and centrifuge  gently to avoid damaging/lysing cells prior to the cell lysis step.</em></strong></li><br/>
</ul>

  <li><strong>Resuspension Buffer</strong></li><br/>
<ol type="1">
  <li>To use  for Lysis Buffer and Wash Buffer, can be stored at room temperature long-term.</li>
<ul type="circle">
<p> 500  &#956;l  1M Tris-HCl, pH 7.5 (final 10 mM)<br />
  100 &#956;l  5 M NaCl (final 10 mM)<br />
  150 &#956;l  1 M MgCl2 (final 3 mM)<br />
  <u>49.25  ml</u> nuclease-free H2O<br />
  50  ml</p>
</ul>
</ol>
  <li><strong>Cell Lysis</strong></li><br/>
<ol type="1">
  <li>Centrifuge  100,000 cells to pellet <u>[NOTE: The person who is sorting and aliquoting  cells will tell you speed/time, to centrifuge usually this is 8,000 xg for 4  minutes]</u>, and then discard supernatant.</li>

  <li>Add 100 &#956;l cold Lysis Buffer, pipet up  and down 3x gently to resuspend cells. <u>[NOTE: Lysis buffer volume should be  scaled to the number of cells; e.g. if you only have 50,000 cells, add 50ul  cold Lysis Buffer]</u></li>
<ul>
<p>97 &#956;l&nbsp;&nbsp; Resuspension Buffer<br/>
  1 &#956;l &nbsp; 10% NP-40 (final 0.1% v/v)<br/>
   1 &#956;l&nbsp;&nbsp; 10%  Tween-20 (final 0.1% v/v)<br />
  <u> 1 </u><u>&#956;</u><u>l</u> &nbsp; 1%  Digitonin (final 0.01% v/v)<br/>
  100 &#956;l</p>
</ul>
  <li>Incubate  on ice x3 minutes.</li>
  <li>Add 1 ml  Wash Buffer, invert tube 3 times gently.</li>
<ul>
<p>990 &#956;l  Resuspension Buffer<br />
  <u>10 </u><u>&#956;</u><u>l</u> 10%  Tween-20 (final 0.1% v/v)<br />
  1 ml</p>
</ul>
  <li>Centrifuge  at 500 xg for 10 minutes at 4oC.</li>

  <li>Discard  supernatant (cytoplasm), keep pellet (nuclei).</li><br/>
</ol>
  <li><strong>Transposition</strong></li><br/>
<ol type="1">
  <li>While  cells are centrifuging, make transposition reaction mix, using the Nextera DNA  Library Prep Kit.&nbsp; <u>[NOTE:  Transposition reaction mix volume should be scaled to the number of cells; e.g.  if you only have 50,000 cells, add 50ul transposition mix]</u></li>
<ul>
<p>50 &#956;l &nbsp;2X TD Buffer (Tagment DNA Buffer)<br />
  33  &#956;l &nbsp;1X PBS<br />
  1 &#956;l &nbsp;10%  Tween-20 (final 0.1% v/v)<br />
  1 &#956;l &nbsp;1%  Digitonin (final 0.01% v/v)<br />
  5 &#956;l &nbsp;Tn5  Transposase (Tagment DNA Enzyme 1)</br>
<br/><u>10</u><u>&#956;</u><u>l</u> nuclease-free H<sub>2</sub>O<br/>
  100 &#956;l</p>
</ul>

  <li>Add  transposition reaction mix to pellet, pipet up and down 6x gently to resuspend  nuclei.</li>
  <li>Incubate  at 37oC for 30 minutes on thermomixer at 1,000 rpm.</li><br/>
</ol>

  <li><strong>DNA Purification</strong></li><br/>
<ol type="1">
  <li>Isolate  DNA using Qiagen MinElute Reaction Cleanup Kit.</li>
  <li>Elute  DNA in 10 &#956;l EB (Elution Buffer).</li>
<ul>
<strong><em>Note: </em></strong><br />
  <strong><em>OK to store DNA at -20oC at this  point.</em></strong></p><br/><br/>

</ul>
</ol>
  <li><strong>PCR Amplification (Library Generation)</strong></li><br/>
<ol type="1">
<li>Combine the following in a PCR tube for each sample:</li>
<ul>
   10 &#956;l&nbsp;&nbsp;&nbsp; purified  transposed DNA<br />
  10 &#956;l&nbsp;&nbsp;&nbsp; nuclease-free  H<sub>2</sub>O<br />
  2.5 &#956;l&nbsp;&nbsp; Ad1_noMX  primer (25 &#956;M)<br />
  2.5 &#956;l&nbsp;&nbsp; Ad2.* indexing primer (25 &#956;M)<br />
  <u>25 &#956;l</u>&nbsp;&nbsp;&nbsp; NEBNext  High-Fidelity 2X PCR Master Mix<br />
  50 &#956;l</p>
  </ul>
<li>Amplify samples in PCR machine with following program:</li>
<ul>
   72oC&nbsp;&nbsp;&nbsp; 5 minutes &nbsp;&nbsp;&nbsp;<b>x5 cycles</b><br />
   98oC&nbsp;&nbsp;&nbsp; 30 seconds &nbsp;<b>x5 cycles</b><br />
  98oC&nbsp;&nbsp;&nbsp; 10  seconds &nbsp;<b>x5 cycles</b><br />
  63oC&nbsp;&nbsp;&nbsp; 30 seconds &nbsp;<b>x5 cycles</b><br />
   72oC&nbsp;&nbsp;&nbsp; 1 minute &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>x5 cycles</b></p>
</ul>
<li>Remove  tubes from PCR machine and use 5 &#956;l of each partially-amplified library to perform  qPCR to determine how many additional PCR cycles are needed.&nbsp; The goal is to stop amplification well prior  to saturation to avoid variation among samples due to PCR bias.</li>
<ul>
  5 &#956;l&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; partially-amplified library<br />
  3.85 &#956;l nuclease-free  H2O<br />
  0.5 &#956;l&nbsp;&nbsp; Ad1_noMX  primer (25 &#956;M)<br />
  0.5 &#956;l&nbsp;&nbsp; Ad2.* indexing primer (25 &#956;M)<br />
  0.15 &#956;l 100X SYBR Green I<br />
  <u>5 &#956;l</u>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; NEBNext  High-Fidelity 2X PCR Master Mix<br />
  15 &#956;l</p>
</ul>
<li>Perform qPCR using following program:</li>
   98oC&nbsp;&nbsp;&nbsp; 30 seconds <b>x20 cycles</b><br />
 98oC&nbsp;&nbsp;&nbsp; 10  seconds <b>x20 cycles</b><br />
  63oC&nbsp;&nbsp;&nbsp; 30 seconds <b>x20 cycles</b><br />
  72oC&nbsp;&nbsp;&nbsp; 1 minute &nbsp;&nbsp;&nbsp;&nbsp;<b>x20 cycles</b></p>
</ul>

<li>**qPCR  Plate setup:</li>
<ol type="a">
  <li>Well  type: unknown</li>
  <li>Collect  fluorescence data for ROX and FAM</li>
  <li>Reference  dye: ROX</li>
</ol>

<li>**qPCR  Thermal Profile setup:</li>
<ol type="a">
  <li>Pre-Melt/RT  Segment: 1 plateau</li>
  <li>Amplification  Segment: normal 3 skip </li>
  <li>Dissociation/Melt  segment: uncheck this box </li>
</ol>
<li>Plot R  vs Cycle Number.&nbsp; Calculate the number of  additional PCR cycles needed for each sample, by determining the number of  cycles needed to reach 1/3 of the maximum R.</li>
<li>Continue PCR on remaining 45 &#956;l of each partially-amplified library for the appropriate number (N)  of cycles:</li>
<ul>
 98oC&nbsp;&nbsp;&nbsp; 30 seconds <b>N cycles</b><br />
  63oC&nbsp;&nbsp;&nbsp; 30 seconds <b>N cycles</b><br />
 72oC&nbsp;&nbsp;&nbsp; 1 minute &nbsp;&nbsp;&nbsp;&nbsp;<b>N cycles</b></p>
</ul><br/>
</ol>
  <li><strong>&nbsp;Library Purification</strong></li><br/>
<ol type="1">
  <li>Warm  AMPure XP beads to room temperature, and vortex for 15 seconds to resuspend. </li>
  <li>For  double-sided bead purification (to remove primer dimers and large &gt;1,000 bp  fragments):</li>
<ol type="a">
    <li>Transfer  each PCR sample to an epi tube, add 0.5X volume (22.5 &#956;l) AMPure XP beads, pipet up and  down 10x to mix thoroughly.</li>
    <li>Incubate  at room temperature for 10 minutes.</li>
    <li>Place  epi tubes in magnetic rack for 5 minutes.</li>
    <li>Transfer  supernatant to new epi tube.</li>
    <li>Add 1.3X  original volume (58.5  &#956;l) AMPure  XP beads, pipet up and down 10x to mix thoroughly.&nbsp; (This results in a final 1.8X bead  buffer:sample ratio.)</li>
    <li>Incubate  at room temperature for 10 minutes.</li>
    <li>Place  epi tubes in magnetic rack for 5 minutes.</li>
    <li>Discard  supernatant.</li>
    <li>Wash  beads with 200 &#956;l  80% EtOH (freshly made), pipet EtOH over beads 10x, then discard EtOH.</li>
    <li>Leave  tube on magnetic rack with cap open for 10 minutes.</li>
    <li>Ensure  all EtOH is removed.</li>
    <li>Resuspend  beads in 20 &#956;l nuclease-free  H2O, pipet up and down 10x to mix thoroughly.</li>
    <li>Incubate  at room temperature for 2 minutes, then quickly spin epi tube down</li>
    <li>Place  epi tube in magnetic rack for 1-5 minutes.</li>
    <li>Transfer  supernatant to new epi tube.</li>
  </ol>
  <li>Store  purified libraries at -20oC.</li><br/>
</ol>
  <li><strong>Assessing Library Quality</strong></li><br/>
<ol type="1">
  <li>Add 1 &#956;l of each library  to 3 &#956;l nuclease-free  H2O (to make 1:4 dilution).</li>
  <li>Run 1 &#956;l of each diluted  library on an Agilent High Sensitivity DNA Bioanalysis chip.</li>
  <li>Use  1 &#956;l of each diluted library to measure DNA concentration by QuBit. </li><br/>
</ol>

  <li><strong>Sequencing</strong></li><br/>
<ol type="1">
  <li>Use 50  bp paired-end (50PE) sequencing.</li>
  <li>Goal is  to obtain &gt;50 million genomic reads per sample minimum to assess open vs  closed chromatin regions, and &gt;200 million genomic reads per sample to detect  transcription factor binding sites.&nbsp;  Remember that many sequencing reads may map to contaminating  mitochondrial DNA.</li><br/>
</ol>
<p align="center"><strong>Table of  PCR Primers (Illumina/Nextera i5 common adapter and i7 index adapters):</strong></p>
<table border="1" align= "center" cellspacing="0" cellpadding="0" width="0">
  <tr>
    <td width="138" valign="top"><p>Ad1_noMX&nbsp;</p></td>
    <td width="493" valign="top"><p>AATGATACGGCGACCACCGAGATCTACACTCGTCGGCAGCGTCAGATGTG&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.1_TAAGGCGA&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATTCGCCTTAGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.2_CGTACTAG&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATCTAGTACGGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.3_AGGCAGAA&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATTTCTGCCTGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.4_TCCTGAGC&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATGCTCAGGAGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.5_GGACTCCT&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATAGGAGTCCGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.6_TAGGCATG&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATCATGCCTAGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.7_CTCTCTAC&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATGTAGAGAGGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.8_CAGAGAGG&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATCCTCTCTGGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.9_GCTACGCT&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATAGCGTAGCGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.10_CGAGGCTG&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATCAGCCTCGGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.11_AAGAGGCA&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATTGCCTCTTGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.12_GTAGAGGA&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATTCCTCTACGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.13_GTCGTGAT&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATATCACGACGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.14_ACCACTGT&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATACAGTGGTGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.15_TGGATCTG&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATCAGATCCAGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.16_CCGTTTGT&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATACAAACGGGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.17_TGCTGGGT&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATACCCAGCAGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.18_GAGGGGTT&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATAACCCCTCGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.19_AGGTTGGG&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATCCCAACCTGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.20_GTGTGGTG&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATCACCACACGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.21_TGGGTTTC&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATGAAACCCAGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.22_TGGTCACA&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATTGTGACCAGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.23_TTGACCCT&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATAGGGTCAAGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
  <tr>
    <td width="138" valign="top"><p>Ad2.24_CCACTCCT&nbsp;</p></td>
    <td width="493" valign="top"><p>CAAGCAGAAGACGGCATACGAGATAGGAGTGGGTCTCGTGGGCTCGGAGATGT&nbsp;</p></td>
  </tr>
</table>
<p>&nbsp;</p>
<hr/>
<p align="left"><font size="+1"><strong>References</strong></font></p>
<ol type="1">
  <li>Buenrostro JD, Giresi PG, Zaba LC, Chang HY, Greenleaf WJ (2013)  Transposition of native chromatin for fast and sensitive epigenomic profiling  of open chromatin, DNA-binding proteins and nucleosome position. Nature methods  10: 1213-1218</li>
  <li>Buenrostro  JD, Wu B, Chang HY, Greenleaf WJ (2015) ATAC-seq: A Method for Assaying  Chromatin Accessibility Genome-Wide. Current protocols in molecular biology /  edited by Frederick M Ausubel&nbsp; [et al]  109: 21.29.21-21.29.29</li>
  <li>Corces  MR, Trevino AE, Hamilton EG, et al. (2017) An improved ATAC-seq protocol  reduces background and enables interrogation of frozen tissues. Nature methods  14: 959</li>
  <li>Milani  P, Escalante-Chong R, Shelley BC, et al. (2016) Cell freezing protocol suitable  for ATAC-Seq on motor neurons derived from human induced pluripotent stem  cells. Scientific Reports 6: 25474</li>
  <li>Ackermann  AM, Wang Z, Schug J, Naji A, Kaestner KH (2016) Integration of ATAC-seq and  RNA-seq identifies human alpha cell and beta cell signature genes. Molecular  Metabolism 5: 233-244</li>
  <li>Tsompana  M, Buck MJ (2014) Chromatin accessibility: a window into the genome.  Epigenetics &amp; chromatin 7: 33</li>
</ol>
</ol>
</body>
</html>
