<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Untitled Document</title>
</head>

<body>
<div>
  <p><h3 align="center">PROTOCOL: ATAC-SEQ</h3></p>
</div>
<ol type="I">
<p><li><b>Reagents</b></li>
<br />
<ul style="list-style-type:disc">
  <li>1X PBS</li>
  <li>Nuclease-free  H<sub>2</sub>O</li>
  <li>NP-40  10%</li>
  <li>Nextera  DNA Library Prep Kit (Illumina, catalog #  FC-121-1030)</li>
  <li>MinElute  Reaction Cleanup Kit (Qiagen, catalog # 28204)</li>
  <li>Primers  (see table at end of protocol)</li>
  <li>NEBNext  High-Fidelity 2X PCR Master Mix (NEB, catalog # M0541S)</li>
  <li>SYBR  Green I (ThermoFisher, catalog # S7563)</li>
  <li>Agencourt  AMPure XP magnetic beads (Beckman Coulter, catalog # A63880)</li>
  <li>80%  Ethanol (made fresh)</li>
  <li>Agilent High  Sensitivity DNA Bioanalysis Kit (Agilent, catalog # 5067-4626)</li>
  <li>Qubit  dsDNA HS Assay Kit &amp; fluorometer (ThermoFisher, catalog # Q32851)</li></p>
</ul>
<li><strong>Notes</strong></li><br />
<ol type="1">
  <li>The  primary protocol is based on the Buenrostro et al. papers, with the following  modification:</li><br />
  <ul style="list-style-type:disc">
  <li>We found that column  purification of the libraries does not remove primer-dimers (78 bp), and our  libraries often contained large fragments 1,000-10,000 bp in length.&nbsp;</li>
  <li> Therefore, we use magnetic bead purification  rather than column purification of the libraries.</li>
  </ul>
  <br />
  <li>Cells  must be alive, fresh, not frozen.</li><br /></ol>
<li><strong>Cell Lysis</strong></li><br />
<ol type="1">
  <li>Wash  50,000 cells with 50 &#956;l cold 1X PBS in epi tube,  centrifuge to pellet, discard supernatant.</li><br />
  <li>Add 50  &#956;l cold lysis buffer, pipet up  and down to resuspend cells.</li>
  <ul style="list-style-type:circle">
  <li>10 &#956;l &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;1M  Tris&middot;Cl, pH 7.4 (final 10 mM)</li>
  <li>2 &#956;l &nbsp;&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;5 M NaCl (final 10 mM)</li>
  <li>3 &#956;l &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 1 M MgCl<sub>2</sub> (final 3 mM)</li>
  <li>10 &#956;l &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 10%  NP-40 (final 0.1% v/v)</li>
  <li><u>975 </u><u>&#956;</u><u>l</u>&nbsp;&nbsp; &nbsp;&nbsp;&nbsp;nuclease-free H<sub>2</sub>O</li>
  <li>1,000 &#956;l &nbsp;&nbsp;<b>Total</b></li></ul><br />
  <li>Centrifuge at 500 xg for 10 minutes at 4<sup>o</sup>C.</li><br />
<li>Discard supernatant (cytoplasm), keep pellet (nuclei)</li></p></ol><br />
<li><strong>Transposition</strong></li><br />
<ol type="1">
  <li>While  cells are centrifuging, make transposition reaction mix, using the Nextera DNA  Library Prep Kit.&nbsp; Volumes per sample of  50,000 cells:</li>
  <ul style="list-style-type:circle">
  <li>25.0   &#956;l &nbsp;&nbsp;&nbsp; 2X  TD Buffer</li>
  <li>2.5 &#956;l &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Tn5  Transposase</li>
  <li> <u>22.5 &#956;l</u>&nbsp;&nbsp;&nbsp;&nbsp; nuclease-free H<sub>2</sub>O</li>
  <li>50.0 &#956;l&nbsp;&nbsp;&nbsp;&nbsp;<b>Total</b></li><br /></ul>
  <li>Add  transposition reaction mix to pellet, pipet up and down to resuspend nuclei.</li><br />
<li>Incubate at 37<sup>o</sup>C for 30 minutes</li></ol></p>
<li><strong>DNA Purification</strong></li>
<br /><ol type="1">
  <li>Isolate DNA using Qiagen MinElute Reaction Cleanup Kit.</li><br />
  <li>Elute  DNA in 10 &#956;l EB (Elution Buffer).</li><br />
<b>NOTE:</b>&nbsp;OK to  store DNA at -20<sup>o</sup>C at this point.
</ol><br />
<li><strong>PCR Amplification (Library  Generation)</strong> </li>
<br />
<ol type="1">
  <li>Combine  the following in a PCR tube for each sample:</li>
  <ul style="list-style-type:circle">
  <li> 10 &#956;l&nbsp;&nbsp;&nbsp;&nbsp; purified  transposed DNA</li>
  <li> 10 &#956;l&nbsp;&nbsp;&nbsp;&nbsp; nuclease-free  H<sub>2</sub>O</li>
  <li> 2.5 &#956;l&nbsp;&nbsp;&nbsp; Ad1_noMX  primer (25 &#956;M)</li>
  <li> 2.5 &#956;l&nbsp;&nbsp;&nbsp; Ad2.* indexing primer (25 &#956;M)</li>
  <li><u>25 &#956;l</u>&nbsp;&nbsp;&nbsp;&nbsp; NEBNext  High-Fidelity 2X PCR Master Mix</li>
  <li>50 &#956;l &nbsp;&nbsp;&nbsp;&nbsp;<b>Total</b></li>
  </ul><br />
  <li>Amplify samples in PCR machine with following program:</li>
  <ul style="list-style-type:circle">
  <li>72<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 5 minutes&nbsp;&nbsp;&nbsp;&nbsp; <b>x 5 cycles</b></li>
<li>98<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 30 seconds&nbsp;&nbsp; <b>x 5 cycles</b></li>
<li>98<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 10  seconds&nbsp;&nbsp; <b>x 5 cycles</b></li>
 <li>63<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 30 seconds &nbsp; <b>x 5 cycles</b></li>
 <li>72<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 1 minute&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <b>x 5 cycles</b></li></ul><br />
  <li>Remove  tubes from PCR machine and use 5 &#956;l of each partially-amplified library to perform  qPCR to determine how many additional PCR cycles are needed.&nbsp; The goal is to stop amplification well prior  to saturation to avoid variation among samples due to PCR bias.</li><br />
  <ul style="list-style-type:circle">
<li> 5.00 &#956;l&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; partially-amplified library</li>
<li>4.41 &#956;l&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; nuclease-free  H<sub>2</sub>O</li>
<li>0.25 &#956;l&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Ad1_noMX  primer (25 &#956;M)</li>
<li> 0.25 &#956;l&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Ad2.* indexing primer (25 &#956;M)</li>
<li> 0.09 &#956;l&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 100X SYBR Green I</li>
<li> <u>5.00 &#956;l</u>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; NEBNext  High-Fidelity 2X PCR Master Mix</li>
<li> 15.00 &#956;l &nbsp;&nbsp;&nbsp;&nbsp;<b>Total</b></ul><br />
  <li>Perform qPCR using following program:</li>
  <ul style="list-style-type:circle">
  <li>98<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 30 seconds&nbsp;&nbsp; <b>x 20 cycles</b></li>
  <li>98<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 10  seconds&nbsp;&nbsp; <b>x 20 cycles</b></li>
  <li>63<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 30 seconds&nbsp;&nbsp; <b>x 20 cycles</b></li>
  <li>72<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 1 minute &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <b>x 20 cycles</b></li></ul><br />
  <li>Plot R  vs Cycle Number.&nbsp; Calculate the number of  additional PCR cycles needed for each sample, by determining the number of  cycles needed to reach 1/3 of the maximum R.</li><br />
  <li>Continue PCR on remaining 45 &#956;l of each partially-amplified library for the appropriate number (N)  of cycles:</li></ol>
 <ul style="list-style-type:circle"> 
 <li>98<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 30 seconds&nbsp;&nbsp; <b>N cycles</b></li>
 <li>98<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 10  seconds&nbsp;&nbsp; <b>N cycles</b></li>
 <li> 63<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 30 seconds&nbsp;&nbsp; <b>N cycles</b></li>  
 <li> 72<sup>o</sup>C&nbsp;&nbsp;&nbsp;&nbsp; 1 minute&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;<b>N cycles</b></li></ul>
  </p>
  <li><strong>Library Purification:</strong></li><br />
  <ol type="1">
    <li>Warm  AMPure XP beads to room temperature, and vortex to resuspend.</li><br />
    <li>For  single left-sided bead purification (to remove primer dimers):</li><br />
    <ul style="list-style-type:circle">
    <li>Transfer each PCR sample to  an epi tube, add 1.8X volume (81 &#956;l) AMPure XP beads, pipet up and down 10x to mix  thoroughly.</li>
    <li>Incubate at room  temperature for 10 minutes.</li>
    <li>Place epi tubes in magnetic  rack for 5 minutes.</li>
    <li>Discard supernatant.</li>
    <li>Wash beads with 200 &#956;l 80% EtOH  (freshly made), pipet EtOH over beads 10x, then discard EtOH.</li>
    <li>Leave tube on  magnetic rack with cap open for 10 minutes.</li>
    <li>Ensure all EtOH is  removed.</li>
    <li>Resuspend beads in 20  &#956;l nuclease-free  H<sub>2</sub>O, pipet up and down 10x to mix thoroughly.</li>
    <li>Place epi tube in magnetic  rack for 1-5 minutes.</li>
    <li>Transfer supernatant to new  epi tube.</li><br /></ul>
    <li>For  double-sided bead purification (to remove primer dimers and large &gt;1,000 bp  fragments):</li>
    <br />
    <ul style="list-style-type:circle">
    <li>Transfer each PCR sample to  an epi tube, add 0.5X volume (22.5 &#956;l) AMPure XP beads, pipet up and down 10x to mix  thoroughly.</li>
    <li>Incubate at room  temperature for 10 minutes.</li>
  <li>Place epi tubes in magnetic  rack for 5 minutes.</li>
   <li>Transfer supernatant to new  epi tube.</li>
    <li>Add 1.3X original volume  (58.5  &#956;l) AMPure  XP beads, pipet up and down 10x to mix thoroughly.&nbsp; (This results in a final 1.8X bead  buffer:sample ratio.)</li>
    <li>Incubate at room  temperature for 10 minutes.</li>
    <li>Place epi tubes in magnetic  rack for 5 minutes.</li>
    <li>Discard supernatant.</li>
    <li>Wash beads with 200 &#956;l 80% EtOH  (freshly made), pipet EtOH over beads 10x, then discard EtOH.</li>
    <li>Leave tube on  magnetic rack with cap open for 10 minutes.</li>
    <li>Ensure all EtOH is  removed.</li>
    <li>Resuspend beads in 20  &#956;l nuclease-free  H<sub>2</sub>O, pipet up and down 10x to mix thoroughly.</li>
   <li>Place epi tube in magnetic  rack for 1-5 minutes.</li>
    <li>Transfer supernatant to new  epi tube.</li></ul><br />
    <li>Store  purified libraries at -20<sup>o</sup>C.</li></ol><br />
  <li><strong>Assessing Library Quality:</strong></li><br />
  <ol type="1">
      <li>Add 1 &#956;l of each library  to 3 &#956;l nuclease-free  H<sub>2</sub>O (to make 1:4 dilution).</li>
    <li>Run 1 &#956;l of each diluted  library on an Agilent High Sensitivity DNA Bioanalysis chip.</li>    <li>Use 1 &#956;l of each diluted library to measure DNA concentration by QuBit.</li></ol><br />
   
  <li><strong>Sequencing:</strong></li> <br />
  
  Use 50  bp paired-end (50PE) sequencing to obtain &gt;50 million reads per sample  minimum to assess open vs closed chromatin regions, and &gt;200 million reads  per sample to detect transcription factor binding sites.</p>

<strong><u><br clear="all" />
</u></strong>
<p><li><strong>Table of PCR Primers (based  on TruSeq indices):</strong></li><br /></p>
  <img src="/assets/workflow/Islet Molecular Phenotyping Studies/Table ATAC.jpg" width="623" height="555" style="vertical-align:middle"/></p><br />
<p><li><strong>References:</strong></li></p>
<ol type="1">
  <li> Ackermann AM, Wang Z, Schug J, Naji A,  Kaestner KH (2016) Integration of ATAC-seq and RNA-seq identifies human alpha  cell and beta cell signature genes. Molecular Metabolism 5: 233-244</li><br />
  <li> Buenrostro JD, Giresi PG, Zaba LC,  Chang HY, Greenleaf WJ (2013) Transposition of native chromatin for fast and  sensitive epigenomic profiling of open chromatin, DNA-binding proteins and  nucleosome position. Nature methods 10: 1213-1218</li><br />
  <li> Buenrostro JD, Wu B, Chang HY,  Greenleaf WJ (2015) ATAC-seq: A Method for Assaying Chromatin Accessibility  Genome-Wide. Current protocols in molecular biology / edited by Frederick M  Ausubel&nbsp; [et al] 109: 21.29.21-21.29.29</li><br />
  <li> Tsompana M, Buck MJ (2014) Chromatin  accessibility: a window into the genome. Epigenetics &amp; chromatin 7: 33</li>
  </ol>
  </ol>
</body>
</html>
