<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>Untitled Document</title>
<style>
img {
    display: block;
    margin-left: auto;
    margin-right: auto;
}
</style>
</head>

<body>
<img src="/assets/workflow/Islet Physiology Studies/HPAP-logo.jpg" alt="hpap-logo" width="200" height="90" style="vertical-align:middle"/>
<h3 align="center"><strong>Whole  Genome Bi-sulphite Sequencing (WGBS)</strong></h3>
<p align="center">Kaestner Lab<br />
  Date Revised: July 15, 2020</p>
<ol type="I">
  <li><strong>Fragmentation </strong></li><br/>
 <ol type="1">
  <li>Covaris Fragmented  gDNA: 55ng. Fragmentation program: 200bp as the Covaris recommended in 130ul in  the low TE buffer by using snap-cap microtube (Covaris, cat#520045)</li><br/>
</ol>
  <li><strong>Bisulfite by  using Ultralow Methylation-seq with TrueMethl oxBS (NuGEN, CAT#0541) Only use  the indicated mock-oxidation steps in the following</strong></li><br/>
<ol type="1">
 <li>Book thermo  cycler and qPCR macine</li>
</ol>
 
<p align="center"><strong>Table  3. Thermal Cycler Programming</strong></p>
<table border="1" align="center" cellspacing="0" cellpadding="0">
  <tr>
    <td bgcolor="#9ebddc" colspan="2">
      <strong>END REPAIR </strong></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>VOLUME </strong></p></td>
  </tr>
  <tr>
    <td><p><strong>Program 1 </strong> <br />
      End Repair </p></td>
    <td><p align="center">25 °C – 30 min, 70 °C – 10 min, hold    at 4 °C </p></td>
    <td align="center"><p>16 μL </p></td>
  </tr>
  <tr>
    <td bgcolor="#9ebddc" colspan="2"><p><strong>ADAPTOR LIGATION </strong></p></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>VOLUME </strong></p></td>
  </tr>
  <tr>
    <td><p><strong>Program 2 </strong> <br />
      Ligation </p></td>
    <td><p align="center">25 °C – 30 min, 70 °C – 10 min, hold    at 4 °C </p></td>
    <td><p align="center">31 μL </p></td>
  </tr>
  <tr>
    <td bgcolor="#9ebddc" colspan="2"><p><strong>FINAL REPAIR </strong></p></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>VOLUME </strong></p></td>
  </tr>
  <tr>
    <td><p><strong>Program 3 </strong> <br />
      Final Repair </p></td>
    <td><p align="center">60 °C – 10 min, hold at 4 °C </p></td>
    <td><p align="center">40 μL </p></td>
  </tr>
  <tr>
    <td bgcolor="#9ebddc" colspan="2"><p><strong>OXIDATION </strong></p></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>VOLUME </strong></p></td>
  </tr>
  <tr>
    <td><p><strong>Program 4 </strong> <br />
      Denaturation </p></td>
    <td><p align="center">37 °C – 5 min, hold at 25 °C </p></td>
    <td><p align="center">10 μL </p></td>
  </tr>
  <tr>
    <td bgcolor="#9ebddc" colspan="2"><p><strong>BISULFITE CONVERSION </strong></p></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>VOLUME </strong></p></td>
  </tr>
  <tr>
    <td><p><strong>Program 5 </strong> <br />
      Bisulfite Conversion </p></td>
    <td><p align="center">95 °C – 5 min, 60 °C – 20 min, 95 °C    – 5 min, 60 °C – 40 min, 95 °C – 5 min, 60 °C – 45 min, hold at 20 °C </p></td>
    <td><p align="center">40 μL </p></td>
  </tr>
  <tr>
    <td bgcolor="#9ebddc" colspan="2"><p><strong>AMPLIFICATION </strong></p></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>VOLUME </strong></p></td>
  </tr>
  <tr>
    <td><p><strong>Program 6 </strong> <br />
      Library Amplification </p></td>
    <td><p align="center">95 °C – 2 min, N(95 °C – 15 s, 60 °C    – 1 min, 72 °C – 30 s), hold at 10 °C </p></td>
    <td></td>
  </tr>
</table><br/>
<ul>
<p><strong><em>Note: </em></strong><br />
  <strong><em>Extra  order: Evagreen</em></strong><br />
  <strong><em>Check  every box, make sure things are ready </em></strong><br />
  <strong><em>Protocol-Day0:  prepare all the material, check all the material, fragment DNA, run  bioanalyzer, put sample in -20C</em></strong><br />
  <strong><em>Protocol-Day1:  until bisulfate, hold on +20C</em></strong></p>
</ul>
<br/>
<li><strong>Fragmented  DNA Purification </strong></li><br/>
<ol type="1">
  <li>Choose a nucleic acid column-based  purification system that allows small volume elution, such as the MinElute® Reaction  Cleanup Kit (QIAGEN, Cat. #28204). Alternatively, you may use the Agencourt  bead- based purification protocol detailed below, which is provided for your  convenience. This protocol is designed to work with a starting volume of 50 μL  of fragmented DNA. If your starting volume is different, change the volume of  beads added in Step 5 to 1.8 times the volume of your fragmented DNA. </li>
  <li>Ensure  the Agencourt beads and Nuclease-free Water (D1) have completely reached room  tempera- ture before proceeding. </li>
  <li>Resuspend  beads by vortexing the tube. Ensure beads are fully resuspended before adding  to sample. After resuspending, do not spin the beads. </li>
  <li>Prepare  a fresh 70% ethanol wash solution. </li>
  <li>Briefly  spin down samples. </li>
  <li>Add  90 μL (1.8 volumes) of bead suspension to each reaction. Mix thoroughly by  pipetting 10 times. </li>
  <li>Incubate  at room temperature for 10 minutes. </li>
  <li>Transfer  the tubes to the magnet and let stand 5 minutes to completely clear the  solution of beads. </li>
  <li>Carefully  remove the binding buffer and discard it </li><br/>
<ul>
  <strong><em>Note:</em></strong><br/>
    <strong><em>The beads should not  disperse; instead, they will stay on the walls of the tubes. Significant loss  of beads at this stage will impact the amount of purified DNA, so ensure beads  are not removed with the binding buffer or the wash.</em></strong>
</ul><br/>
  <li>With  the tubes still on the magnet, add 200 μL of freshly prepared 70% ethanol and  allow to stand for 30 seconds. </li>
  <li>Remove  the 70% ethanol wash using a pipette. </li>
  <li>Repeat  steps 9 and 10 for a total of two 70% ethanol washes. </li>
  <li>Air  dry the beads on the magnet for 5–10 minutes. Inspect each tube carefully to  ensure that all the ethanol has evaporated. </li>
  <li>Remove  the tubes from the magnet. </li>
  <li>Add  14 μL room temperature of 1X low-EDTA TE buffer or Nuclease-free Water (green:  D1) to the dried beads. Mix thoroughly to ensure all the beads are resuspended. </li>
  <li>Transfer  the tubes to the magnet and let stand for 5 minutes to completely clear the  solution of beads. </li>
  <li>Carefully  remove 13 μL of the eluate, ensuring as few beads as possible are carried over;  transfer to a fresh set of PCR tubes and place on ice. </li><br/>
<ul>
<p><strong><em>Note:</em></strong><br />
<strong><em>Optional stopping point: Store  samples at –20 °C. </em></strong></p>
<br/>
</ul>
</ol>

<li><strong>End Repair</strong></li>

<p align="center"><strong>Table 5. End Repair Master Mix</strong></p>
<table border="1" align="center" cellspacing="0" cellpadding="0">
  <tr>
    <td bgcolor="#9ebddc">
      <p align="center"><strong>REAGENT</strong></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>END    REPAIR BUFFER MIX (BLUE: ER1 </strong>ver <strong>8)</strong></p></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>END    REPAIR ENZYME MIX (BLUE: ER2 </strong>ver <strong>4)</strong></p></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>END    REPAIR ENHANCER (BLUE: ER3 </strong>ver <strong>2)</strong></p></td>
  </tr>
  <tr>
    <td><p align="center">STORAGE </p></td>
    <td><p align="center">–20    °C </p></td>
    <td><p align="center">–20    °C </p></td>
    <td><p align="center">–20    °C </p></td>
  </tr>
  <tr>
    <td><p align="center">1X REACTION VOLUME </p></td>
    <td><p align="center">2.0    μL </p></td>
    <td><p align="center">0.5    μL </p></td>
    <td><p align="center">0.5    μL </p></td>
  </tr>
</table><br/>
<ol type="1">
  <li>Thaw  ER1 at room temperature. Mix by vortexing, spin down and place on ice. </li>
  <li>Spin  down ER2 and ER3 and place on ice. </li>
  <li>Obtain  13 μL of fragmented DNA sample (10–300 ng) from the DNA Purification protocol. </li><br/>
<ul>
<strong>Note:  <br />
  <em>To  interrogate 5hmC, split the sample into two 13 μL aliquots to perform parallel  library preparations with oxBS and MOCK oxBS workflows. The aliquots can be  diluted using Nuclease-free Water or 1X low-EDTA TE buffer. </em></strong>
</ul><br/>
  <li>Prepare  a master mix by combining ER1, ER2 and ER3 in a 0.5 mL capped tube, according  to the volumes shown in Table 5. </li>
  <li>Add  3 μL of End Repair Master Mix to each sample tube for a total of 16 μL. </li>
  <li>Mix  by pipetting, cap and spin tubes and place on ice. </li>
  <li>Place  tubes in a pre-warmed thermal cycler programmed to run Program 1 (End Repair;  see Table 3): </li>
 <ol type="a">
<li>25 °C – 30 min</li>
<li>70 °C – 10 min</li>
<li>hold at 4 °C</li>
</ol>
  <li>Remove  tubes from the thermal cycler, spin to collect condensation and place on ice. </li>
</ol><br/>

<li><strong>Adaptor Ligation </strong></li><br/>
<ol type="1">
  <li>Remove  Agencourt beads from 4 °C. Place on the bench top to reach room temperature for  use in the next step. </li>
<br/>
<p align="center"><strong>Table 6. Ligation Master Mix</strong></p>
<table border="1" cellspacing="0" cellpadding="0">
  <tr>
    <td bgcolor="#9ebddc">
      <p align="center"><strong>REAGENT</strong></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>NUCLEASE-FREE    WATER (GREEN: D1)</strong></p></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>LIGATION    BUFFER MIX (YELLOW: L1 </strong>ver <strong>4)</strong></p></td>
    <td bgcolor="#9ebddc"><p align="center"><strong>LIGATION    ENZYME MIX (YELLOW: L3 </strong>ver <strong>4)</strong></p></td>
  </tr>
  <tr>
    <td><p align="center">STORAGE </p></td>
    <td><p align="center">---</p></td>
    <td><p align="center">-20    °C</p></td>
    <td><p align="center">-20    °C</p></td>
  </tr>
  <tr>
    <td><p align="center">1X REACTION VOLUME </p></td>
    <td><p align="center">4.5    μL</p></td>
    <td><p align="center">6    μL</p></td>
    <td><p align="center">1.5μL</p></td>
  </tr>
</table><br/>

  <li>Spin down L3 and place  on ice. </li>
  <li>Thaw adaptor plate on  ice, spin down, and return to ice. </li>
  <li>Thaw L1 at room  temperature. Mix by vortexing, spin down and place on ice. Add adaptors to each  sample as follows: </li>
<ol type="a">
      <li>If using adaptors from  32-plex Adaptor Plate (S02694), add 3 μL of the appropriate Ligation Adaptor  Mix to each sample. Mix thoroughly by pipetting. </li>
      <li>If using adaptors from  96-plex Adaptor Plate (S02169), add the entire 16 μL of sample to the  appropriate adaptor well, mix well by pipetting, then transfer the entire  sample to a PCR tube </li>
</ol>
<ul>
<br/>
<p><strong><em>Note:<br/>In order  to multiplex the oxBS and MOCK oxBS processed samples, unique  indexes are required.</em></strong></p>
</ul>
<br/>
  <li>Prepare a master mix  by combining D1, L1 and L3 in a 0.5 mL capped tube, according to the vol- umes  shown in Table 6. Mix by pipetting slowly, without introducing bubbles, spin  and place on ice. Use the master mix immediately. </li>
<ul>
<br/>
<p><strong><em>Note:The L1  Ligation Buffer Mix is very viscous. Pipet this reagent slowly and mix  thoroughly.</em></strong></p>
</ul>
<br/>
  <li>Add 12 μL Ligation  Master Mix to each reaction for a total of 31 μL. Mix thoroughly by pipetting  slowly and gently, spin down and place on ice. Proceed immediately with the  incubation. </li>
  <li>Place tubes in a  pre-warmed thermal cycler programmed to run Program 2 (Ligation; see Table 3):</li>
  <ol type="a">
  <li>25 °C – 30 min</li>
  <li>70 °C – 10 min</li>
  <li>hold at 4 °C</li>
  </ol>
    <li>Remove the tubes from the thermal cycler, spin to collect condensation and  place on ice.</li>
</ol><br/>
  
<li><strong>Post-Ligation Purification </strong></li><br/>
 <ol type="1">
  <li>Ensure  the Agencourt beads and D1 have completely reached room temperature before  proceeding. </li>
  <li>Resuspend  the beads by vortexing the tube. Ensure the beads are fully resuspended before  adding to the sample. After resuspending, do not spin the beads. </li>
  <li>At  room temperature, add 45 μL (1.5 volumes) of the bead suspension to each  reaction. Mix thor- oughly by pipetting 10 times. </li>
  <li>Incubate  at room temperature for 10 minutes. </li>
  <li>Transfer  the tubes to the magnet and let stand 5 minutes to completely clear the solution  of beads. </li>
  <li>Carefully  remove the binding buffer and discard it. </li>
<br/>
<ul>
<p><strong><em>Note:<br/>The beads should not  disperse; instead, they will stay on the walls of the tubes. Significant loss  of beads at this stage will impact the amount of purified DNA, so ensure beads  are not removed with the binding buffer or the wash. </em></strong></p>
</ul><br/>
  <li>With  the tubes still on the magnet, add 200 μL of freshly prepared 70% ethanol and  allow to stand for 30 seconds. </li>
  <li>Remove  the 70% ethanol wash using a pipette. </li>
  <li>Repeat  steps 7 and 8 for a total of two 70% ethanol washes. </li>
<br/>
<ul>
<p><strong><em>Note:<br/>With the final wash,  it is critical to remove as much of the ethanol as possible. Use at least two  pipetting steps and allow excess ethanol to collect at the bottom of the tubes  after removing most of the ethanol in the first pipetting step.</em></strong></p>
</ul><br/>
  <li>Air dry the beads on  the magnet for 5–10 minutes. Inspect each tube carefully to ensure that all the  ethanol has evaporated. It is critical that all residual ethanol be removed  prior to continuing. </li>
  <li>Remove the tubes from  the magnet. </li>
  <li>Add 16 μL room  temperature of 1X low-EDTA TE buffer or D1 to the dried beads. Mix thoroughly  to ensure all the beads are resuspended. </li>
  <li>Transfer the tubes to  the magnet and let stand for 5 minutes to completely clear the solution of  beads. </li>
  <li>Carefully remove 15  μL of the eluate, ensuring as few beads as possible are carried over; transfer  to a fresh set of PCR tubes and place on ice. </li>
</ol><br/>
<li><strong>Final Repair </strong></li><br/>
<ol type="1">
   <li>Remove  the TrueMethyl oxBS Module Magnetic Bead Solution, Binding Buffer 1 and Binding  Buffer 2 from 4 °C and place at room temperature for use in the next step. </li>
 
 <p align="center"><strong>Table 7. Final Repair  Master Mix</strong></p>
 <table border="1" align="center" cellspacing="0" cellpadding="0" width="0">
   <tr>
     <td bgcolor="#9ebddc">
       <p align="center"><strong>REAGENT</strong></td>
     <td bgcolor="#9ebddc" width="263"><p align="center"><strong>FINAL REPAIR BUFFER MIX (PURPLE: FR1 </strong>ver <strong>4)</strong></p></td>
     <td bgcolor="#9ebddc" width="263"><p align="center"><strong>FINAL REPAIR ENZYME MIX (PURPLE: FR2)</strong></p></td>
   </tr>
   <tr>
     <td><p align="center">STORAGE </p></td>
     <td width="263"><p align="center">–20    °C</p></td>
     <td width="240"><p align="center">–20    °C</p></td>
   </tr>
   <tr>
     <td><p align="center">1X REACTION VOLUME </p></td>
     <td width="263"><p align="center">4.5 μL</p></td>
     <td width="263"><p align="center">0.5 μL</p></td>
   </tr>
 </table>
 <br/>
   <li>Spin  down FR2 and place on ice. </li>
   <li>Thaw  FR1 at room temperature. Mix by vortexing, spin down and place on ice. </li>
   <li>Prepare  a master mix by combining FR1 and FR2 in a 0.5 mL capped tube, according to the  volumes shown in Table 7. </li>
   <li>Add  5 μL of the Final Repair Master Mix to each sample tube for a total of 20 μL. </li>
   <li>Mix  by pipetting, cap and spin tubes and place on ice. </li>
   <li>Place  the tubes in a pre-warmed thermal cycler programmed to run Program 3 (Final  Repair; see Table 3): </li>
   <li>60  °C – 10 min, hold at 4 °C </li>
   <li>Remove  the tubes from the thermal cycler, spin to collect condensation and place on  ice. </li>
 <br/>
 <strong><em>Note:<br/>Optional stopping point: Store  samples at –20 °C.</em></strong><br/>
 <br/>
   <li>If  using the True Methyl oxBS Module, continue with section H. DNA Purification  and Denaturation. For alternative bisulfite conversion methods, perform  bisulfite conversion according to the manufac- turer recommendations and  proceed to protocol section <strong>L. Library Amplification Optimization with qPCR</strong>.</li>
</ol>
<br/>
<ul>
  <p><strong><em>Note:<br/>Bisulfite conversion is critical for successful library  amplification</em></strong></p>
</ul><br/>

<li><strong>DNA Purification and Denaturation </strong></li><br/>
<ol type="1">
   <li>Remove  the TrueMethyl oxBS Module Oxidant Solution from -20 °C and thaw on ice for use  in the next step. </li>
<br/>
<strong><em>Note:<br/> The Oxidant Solution is  light-sensitive. Keep protected from light.</em></strong><br/>
<br/>

 <p align="center"><strong>Table  8. Magnetic Bead Binding Solution 1 Master Mix</strong></p>
 <table border="1" cellspacing="0" cellpadding="0" align="center">
   <tr>
     <td bgcolor="#9ebddc"><p align="center"><strong>REAGENT </strong></p></td>
     <td bgcolor="#9ebddc"><p align="center"><strong>BINDING BUFFER 1 </strong></p></td>
     <td bgcolor="#9ebddc"><p align="center"><strong>MAGNETIC BEAD    SOLUTION </strong></p></td>
   </tr>
   <tr>
     <td><p align="center">STORAGE </p></td>
     <td><p align="center">4 °C </p></td>
     <td><p align="center">4 °C </p></td>
   </tr>
   <tr>
     <td><p align="center">1X REACTION VOLUME </p></td>
     <td><p align="center">120 μL </p></td>
     <td><p align="center">2.4 μL </p></td>
   </tr>
 </table><br/>

   <li>Remove  Acetonitrile, Ultra Pure water, and Denaturing Solution from storage and place  on bench top. </li>
   <li>Prepare a fresh stock  of 80% acetonitrile, using the Ultra Pure water provided with the kit. Mix by  vortexing or inversion and place at room temperature. </li>
   <li>Ensure Magnetic Bead  Solution and Binding Buffer 1 have reached room temperature before proceeding. </li>
   <li>Mix Binding Buffer 1  by inversion until homogenized. </li>
   <li>Vortex Magnetic Bead  Solution until homogenized. </li>
   <li>Prepare a master mix  of Magnetic Bead Binding Solution 1 (MBBS1) as directed in Table 8. </li>
<br/>
<ul>
 <p><strong><em>Note:<br/>MBBS1 should be  prepared fresh on the day of use. Do not store for longer than 1</em></strong></p>
</ul><br/>
   <li>Vortex MBBS1 master  mix thoroughly to ensure the beads are homogenized in solution. </li>
   <li>At room temperature,  add 30 μL water of Ultra Pure water to each sample for a total of 50 μL. </li>
   <li>Add 100 μL of MBBS1  master mix to each 0.2 mL tube containing 50 μL NGS library for a total of 150  μL. Mix by pipetting and centrifuge briefly. </li>
   <li>Incubate at room  temperature for 20 min. </li>
   <li>Transfer tubes to a  magnetic separation plate and incubate at room temperature for 5 minutes to  completely clear the solution of beads. </li>
   <li>Keeping the tubes on  the magnet, carefully remove the supernatant and discard it. </li>
   <li>With the tubes still  on the magnet, carefully add 200 μL of 80% Acetonitrile wash to the tubes with-  out disturbing the bead pellet. </li>
   <li>Remove and discard  the 200 μL 80% Acetonitrile wash, carefully avoiding aspiration of the bead  pellet. </li>
   <li>Repeat Steps 13 and  14 twice to perform 3 x 200 μL 80% Acetonitrile washes in total. Remove as much  of the final wash as possible. </li>
<br/>
<ul>
 <p><strong><em>Note:<br/>Ensure the tubes are  dry before continuing the protocol. If the tubes aren&rsquo;t dry after 5 minutes,  incubate for a longer period of time.</em></strong></p>
</ul><br/>
   <li>Air dry the bead  pellets for 5 minutes at room temperature, leaving the lids of the tubes open. </li>
   <li>Remove the tubes from  the magnet. </li>
   <li>Add 10 μL of  Denaturing Solution directly onto the bead pellet. Mix thoroughly to ensure all  beads are resuspended. </li>
   <li>Centrifuge briefly to  collect solution at bottom of the tubes. </li>
   <li>Place  the tubes in a pre-warmed thermal cycler programmed to run Program 4  (Denaturation; see Table 3): </li>
   <ol type="a">
     <li>37  °C – 5 min</li>
     <li>hold at 25 °C </li>
   </ol>
   <li>Remove the tubes from  the thermal cycler, spin to collect condensation and transfer to the magnet. </li>
   <li>Incubate at room  temperature for 2 minutes. </li>
   <li>Carefully remove 9 μL  of the eluate, ensuring as few beads as possible are carried over, transfer to  a fresh 1.5 mL microcentrifuge tube and place at room temperature. </li>
   <li>Remove  Ultra Pure water from storage and place on bench top. </li>
   <li>Set  a heat block to 40 °C. </li>
   <li>Prepare  individual oxidation and mock oxidation reactions as follows: </li>
<ol type="a">
   <li><strong>Do not do:</strong> For each sample to  be processed through the oxBS workflow, add 1 μL of oxidant solution to 9 μL of  DNA for a total of 10 μL. </li>
   <li><strong>Only do:</strong> For each sample to  be processed through the MOCK oxBS workflow, add 1 μL of Ultra Pure water to 9  μL of DNA for a total of 10 μL. </li>
</ol>
   <li>Mix  reactions by vortexing and centrifuge briefly.</li>
   <li>Place  tubes in heat block and incubate for 10 min at 40 °C. </li>
   <li>Centrifuge  reactions at 14000 x g for 10 minutes at room temperature to pellet any black  precipitate. MOCK treated samples will remain clear and will not have any black  precipitate. </li>
<br/>
<ul>
 <p><strong><em>Note:<br/>In samples treated with oxidant solution, the color of the oxidation reaction  should remain orange after the 10 minute centrifugation, indicating a successful  oxidation. If the solution turns any color other than orange, please see topic XVI below. </em></strong></p>
</ul><br/>
   <li>Transfer  the orange supernatant to a fresh 0.2 mL PCR tube and place at room  temperature. Proceed immediately to the next step. </li><br/>
 
<ul type="disc">
<strong><em>Note:</em></strong><br/>
<li><strong><em>Take care not to carry any black precipitate over as this  could inhibit downstream steps. </em></strong></li>
<li><strong><em>Do not place the oxidized samples on ice to cool as this may  cause the solution to precipitate. </em></strong></li>
</ul>
</ol>
<br/>
<li><strong>Bisulfite Conversion </strong></li><br/>
<ol type="1">
<li>Set  a heat block or heated orbital incubator to 60 °C. </li>
<li>Remove  Bisulfite Diluent and Bisulfite Reagent aliquots from storage and place on  bench top. Remove 1 aliquot of Bisulfite Reagent for every 20 reactions to be  processed. </li>
<li>Prepare  Bisulfite Reagent Solution by adding 700 μL of Bisulfite Diluent to each  aliquot of Bisulfite Reagent. </li>
<br/>
<ul>
<p><strong><em>Note:<br/>Each aliquot of  Bisulfite Reagent Solution is sufficient for up to 20 samples. A fresh aliquot of solution should be  prepared each time the kit is used and disposed  of immedi- ately after use.</em></strong></p>
</ul><br/>
  <li>Seal  the lid of each aliquot with Bisulfite Reagent Solution tightly. </li>
  <li>Incubate  the aliquots of Bisulfite Reagent Solution for 15 min at 60 °C. Vortex  regularly until the Bisulfite Reagent Solution is completely dissolved. </li>
  <li>Spin  down Bisulfite Reagent Solution briefly and place at room temperature. </li>
  <li>Ensure  oxidized DNA samples from previous step are at room temperature before  proceeding. </li>
  <li>Prepare  Bisulfite Conversion Reaction mix by adding 30 μL of Bisulfite Reagent Solution  to each 10 μL of DNA for a total of 40 μL. Ensure that each sample pair being  processed through the oxBS and MOCK oxBS workflow is treated with the same  aliquot of Bisulfite Reagent Solution. Mix by pipetting, spin down and place at  room temperature. </li>
  <li>Mix  by pipetting, spin down and place at room temperature. </li><br/>

<br/>
<ul type="disc">
<strong><em>Note:</em></strong><br/>
  <li><strong><em>If  the Bisulfite Reagent Solution precipitates, return to 60 °C until dissolved. </em></strong></li>
  <li><strong><em>Samples  treated with the oxidant solution may turn light gray in color. </em></strong></li><br/>
</ul><br/>

  <li>Place the tubes in a  pre-warmed thermal cycler programmed to run Program 5 (Bisulfite Conversion,  see Table 3): </li>
<ol type="a">
    <li>95 °C – 5 min, 60 °C  – 20 min</li>
    <li>95 °C – 5 min, 60 °C  – 40 min</li>
    <li>95 °C – 5 min, 60 °C  – 45 min </li>
    <li>hold at 20 °C </li>
  </ol>
<br/>
<ul>
<p><strong><em>Note: </em></strong><br />
  <strong><em>Optional stopping point: You may  hold samples at room temperature (+20°C) for up to 16 hours. Do not store below +20 °C. </em></strong></p>
</ul><br/>
  <li>Once  the bisulfite conversion is complete, centrifuge samples briefly to collect  solution at bottom of the tubes. </li>
  <li>Transfer samples to  1.5 mL tubes and centrifuge for 10 min at 14000 x g. </li>
  <li>Continue to K. Bisulfite-Converted DNA Desulfonation and Purification while the samples are in  the centrifuge. </li>
</ol><br/>
<li><strong>Bisulfite-Converted DNA Desulfonation and Purification </strong></li><br/>
<ol type="1">
  <li>Remove  Desulfonation Buffer, Binding Buffer 2, Magnetic Bead Solution and Elution  Buffer from stor- age and place at room temperature for a minimum of 30 minutes  before use. </li>

<p align="center"><strong>Table  9. Magnetic Bead Binding Solution 2 Master Mix</strong></p>

  <table border="1" align="center" cellspacing="0" cellpadding="0">
    <tr>
      <td bgcolor="#9ebddc">
        <p align="center"><strong>REAGENT</strong></p></td>
      <td bgcolor="#9ebddc"><p align="center"><strong>BINDING BUFFER 2</strong></p></td>
      <td bgcolor="#9ebddc"><p align="center"><strong>MAGNETIC BEAD SOLUTION</strong></p></td>
    </tr>
    <tr>
      <td><p align="center">STORAGE </p></td>
      <td><p align="center">4    °C </p></td>
      <td><p align="center">4    °C </p></td>
    </tr>
    <tr>
      <td><p align="center">1X REACTION VOLUME </p></td>
      <td><p align="center">200    μL </p></td>
      <td><p align="center">2.4    μL </p></td>
    </tr>
  </table><br/>

  <li>Prepare  a fresh stock of 70% Ethanol. Mix by vortexing or inversion. </li>
  <li>Mix  Binding Buffer 2 by inversion until homogenized. </li>
  <li>Vortex  Magnetic Bead Solution until homogenized. </li>
  <li>Prepare  a master mix of Magnetic Bead Binding Solution 2 (MBBS2) as directed in Table  10. </li><br />
<ul type="disc">
<strong><em>Note:</em></strong>
  <li><strong><em>MBBS2 should be prepared fresh on the day of use. Do not  store for longer than 1 week. </em></strong></li>
  <li><strong><em>MBBS2 is a viscous solution. Pipet this reagent slowly and  mix thoroughly. Ensure that MBBS2 and the MBBS2-sample mix are well-mixed. </em></strong></li><br/>
</ul>
  <li>Transfer  40 μL of the supernatant to a fresh set of 0.2 mL PCR tubes. Avoid disturbing  the pellet in the oxBS-treated samples. </li>
  <li>Vortex  MBBS2 thoroughly to ensure the solution is homogenous before aliquoting.  </li>
  <li>Carefully  add 160 μL of MBBS2 to each tube containing 40 μL bisulfite converted sample  for a total of 200 uL. Mix thoroughly by pipetting slowly and gently, spin down  and place at room temperature. </li>
  <li>Incubate  at room temperature for 5 minutes. </li>
  <li>Centrifuge  briefly to collect solution at bottom of the tubes. </li>
  <li>Place  the tubes onto the magnet and incubate at room temperature for at least 5  minutes to com- pletely clear the solution of beads. </li>
  <li>Carefully  remove the supernatant and discard it.<br />
    13. Remove the tubes from the magnet.<br />
    14. Add 200 μL of 70% Ethanol to each sample tube. Resuspend the beads  completely by pipetting. </li>
  <li>Place  the tubes on the magnet and incubate at room temperature for 5 minutes to  completely clear the solution of beads. </li>
  <li>Carefully  remove the 70% Ethanol wash and discard it. Remove as much of the wash as  possible. </li>
  <li>Remove  samples from the magnet. </li>
  <li>Add  200 μL of Desulfonation Buffer with EtOH added directly onto the bead pellet.  Resuspend the beads completely by pipetting. </li>
<br/>
<ul>
<p><strong><em>Note:<br/>Be sure that the  ethanol has been added to the desulfonation buffer, as described in Section III B. Preparation of Desulfonation Buffer for the  TrueMethyl oxBS Module.</em></strong><strong></strong></p>
</ul><br/>
  <li>Close  lids of sample tubes securely and place the tubes into the magnetic separation  rack. Incubate at room temperature for 5 minutes to completely clear the  solution of beads. </li>
  <li>Remove  the tubes from the magnet, open the tubes, and return to the magnet. </li>
  <li>Carefully  remove 200 μL of the Desulfonation Buffer and discard it. Remove as much of the  Desulfonation Buffer as possible without disturbing the bead pellet. </li>
  <li>Remove  the tubes from the magnet. </li>
  <li>Add  200 μL of 70% Ethanol to each sample tube. Resuspend the beads completely by  pipetting. </li>
  <li>Place  the tubes onto the magnet and incubate at room temperature for 5 minutes to  completely clear the solution of beads. </li>
  <li>Remove  the 200 μL 70% Ethanol wash and discard it. </li>
  <li>Repeat  Steps 23-25 to perform 2 x 200 μL 70% Ethanol washes in total. Remove as much  of the final wash as possible. </li>
  <li>Air-dry  the the beads on the magnet for 15 minutes. Inspect each tube carefully to  ensure that all of the ethanol has evaporated. </li>
  <li>Remove  the tubes from the magnet. </li>
  <li>Add  Elution Buffer directly onto the bead pellet and resuspend completely by  pipetting: </li>
 <ol type="a">
      <li>For Library  Amplification Optimzation with qPCR (recommended), resuspend beads in 23 μL  Elution Buffer. </li>
      <li>If qPCR optimization  is not required, resuspend beads in 21 μL Elution Buffer. </li>
</ol>
  <li>Incubate  at room temperature for 5 minutes to elute the TrueMethyl converted DNA from  the beads. </li>
  <li>Centrifuge  briefly to collect sample at bottom of the tubes. </li>
  <li>Place  the tubes onto the magnet and incubate at room temperature for 5 minutes to  completely clear the solution of beads. </li>
  <li>Carefully  transfer eluate into a fresh 0.2 mL tube: </li>
<ol type="a">
      <li>For Library  Amplification Optimzation with qPCR (recommended), transfer 22 μL eluate. </li>
      <li>If qPCR optimization  is not required, transfer 20 μL eluate.</li> <br/>
</ol>
</ol>
<br/>
<li><strong>Library Amplification Optimization with qPCR </strong></li>

<p align="center"><strong>Table 10. Library Amplification qPCR  Master Mix</strong></p>
<table border="1" align="center" cellspacing="0" cellpadding="0">
  <tr>
    <td width="200" bgcolor="#9ebddc"><br />
      <p align="center"><strong>REAGENT</strong></p></td>
    <td width="200" bgcolor="#9ebddc"><p align="center"><strong>AMPLIFICATION    PRIMER MIX (RED: P2 </strong>ver <strong>8)</strong></p></td>
    <td width="200" bgcolor="#9ebddc"><p align="center"><strong>AMPLIFICATION ENZYME MIX (RED: P3 </strong>ver <strong>3)</strong></p></td>
    <td width="200" bgcolor="#9ebddc"><p align="center"><strong>20 x EvaGreen</strong></p></td>
  </tr>
  <tr>
    <td width="119"><p align="center">STORAGE </p></td>
    <td width="184"><p align="center">–20 °C</p></td>
    <td width="184"><p align="center">–20 °C</p></td>
    <td width="78"><p align="center">4 °C</p></td>
  </tr>
  <tr>
    <td width="119"><p align="center">1X REACTION VOLUME </p></td>
    <td width="184"><p align="center">1.0 μL</p></td>
    <td width="184"><p align="center">4.75 μL</p></td>
    <td width="78"><p align="center">0.5 μL</p></td>
  </tr>
</table><br/>
<ol type="1">
  <li>Aliquot 2 μL of each  sample into a fresh set of 0.2 mL tubes. Reserve the remaining 20 μL of sample  on ice. </li>
  <li>Add 14 μL of Low-EDTA  TE Buffer to each sample for a total of 16 μL. Mix well by pipetting, spin down  and place on the bench top. </li>
  <li>Aliquot 3.75 μL of  each sample into an appropriate PCR plate or optically clear strip tubes, in  tripli- cate. Mix well by pipetting, spin down and place on ice. </li>
  <li>Prepare a master mix  by combining P2, P3 and 20x EvaGreen in an appropriately sized capped tube  according to the volumes shown in Table 11. Add P3 at the last moment and mix  well by pipetting, taking care to avoid bubbles. Spin down and place on ice. </li>
  <li>On ice, add 6.25 μL  of Library Amplification qPCR Master Mix to each 3.75 μL of sample, in  triplicate, for a total of 10 μL per replicate. </li>
  <li>Mix well by  pipetting, spin down and place on ice. </li>
  <li>Perform  qPCR with the following cycling conditions: </li>
<ol type="a">
  <li>95 °C – 2 min, 35x(95°C – 15s </li>
  <li>60 °C – 1 min, 72°C – 30s) </li>
 </ol>
  <li>Examine the log  fluorescence vs. cycle number plot from the qPCR system to determine the appro-  priate number of library amplification cycles. </li>
 <ol type="a">
  <li>First, select a cycle  number within the late exponential phase of the amplification plot. In the  example in <strong>Figure 5</strong>, this is 11 cycles the &lsquo;BS Library&rsquo; (blue dotted  vertical line, left) and 14 cycles in the &lsquo;oxBS Library&rsquo; (red dotted vertical  line, right). </li>
  <li>Next, subtract 3 from  the selected cycle to determine the number of PCR cycles to perform in the next  step (M. Library Amplification). This compensates for the 1:8 dilution of  sample used in the qPCR reaction. </li>
</ol>
</ol>
<br/>
  <li><strong>Library Amplification </strong></li>
<br/>
<p align="center"><strong>Table 11. Library  Amplification Master Mix</strong></p>
<table border="1" align="center" cellspacing="0" cellpadding="0" width="0">
  <tr>
    <td bgcolor="#9ebddc"><br />
      <p align="center"><strong>REAGENT</strong></p></td>
    <td width="300" bgcolor="#9ebddc"><p align="center"><strong>AMPLIFICATION    PRIMER MIX (RED: P2 </strong>ver <strong>8)</strong></p></td>
    <td width="300" bgcolor="#9ebddc"><p align="center"><strong>AMPLIFICATION    ENZYME MIX (RED: P3 </strong>ver <strong>3)</strong></p></td>
  </tr>
  <tr>
    <td><p align="center">STORAGE </p></td>
    <td width="300"><p align="center">–20 °C</p></td>
    <td width="300"><p align="center">–20 °C</p></td>
  </tr>
  <tr>
    <td width="300"><p align="center">1X REACTION VOLUME </p></td>
    <td width="300">
      <p align="center">5.0 μL</p>
   
    </td>
    <td width="300">
    <p align="center">25 L</p> 
  </td>
  </tr>
</table><br/>
<ol type="1">
  <li>Remove  Agencourt beads from 4 °C. Place on the bench top to reach room temperature for  use in the next step. </li>
  <li>Thaw  P3 on ice if needed. Spin down and place on ice. </li>
  <li>Thaw  P2 at room temperature. Mix by vortexing, spin and place on ice. </li>
  <li>Prepare  a master mix by combining P2 and P3 in an appropriately sized capped tube  according to the volumes shown in Table 11. Add P3 at the last moment and mix  well by pipetting, taking care to avoid bubbles. Spin down and place on ice. </li>
  <li>On  ice, add 30 μL of Library Amplification Master Mix to each 20 μL of reserved  sample for a total of 50 μL. </li>
  <li>Mix  well by pipetting, spin down and place on ice. </li>
  <li>Place  tubes in a pre-warmed thermal cycler programmed to run Program 6 (Library  Amplification; see Table 3): </li>
<ol type="a">
  <li>95 °C – 2 min, N (95  °C – 15 s, 60 °C – 1 min 72 °C – 30 s)</li>
  <li>hold  at 10 °C </li>
</ol>
  <li>Remove  the tubes from the thermal cycler, spin to collect condensation and place on  ice. </li><br/>
</ol>
  <li><strong>Amplified Library Purification </strong></li><br/>
<ol type="1">
  <li>Ensure  the Agencourt beads have completely reached room temperature before proceeding. </li>
  <li>Resuspend  the beads by inverting and tapping the tube. Ensure the beads are fully  resuspended before adding to samples. After resuspending, do not spin the  beads. </li>
  <li>Add  50 μL (1 volume) of the bead suspension to each reaction. Mix thoroughly by  pipetting 10 times. </li>
  <li>Incubate  at room temperature for 10 minutes. </li>
  <li>Transfer  the tubes to the magnet and let stand 5 minutes to completely clear the  solution of beads. </li>
  <li>Carefully  remove the binding buffer and discard it. </li>
<br/>
<ul>
<p><strong><em>Note:<br/></em></strong><strong><em>The beads should not  disperse; instead, they will stay on the walls of the tubes. Significant loss of beads at this stage  will impact the final yield, so ensure beads are not removed with the binding buffer or the wash.</em></strong><strong></strong></p>
</ul><br/>
  <li>With samples still on  the magnet, add 200 μL of freshly prepared 70% ethanol. </li>
  <li>Remove  the 70% ethanol wash using a pipette. </li>
  <li>Repeat  steps 7 and 8 for a total of two washes. </li>
<br/>
<ul>
<p><strong><em>Note:<br/></em></strong><strong><em>With the final wash,  it is critical to remove as much of the ethanol as possible. Use at least two  pipetting steps and allow excess ethanol to collect at the bottom of the tubes  after removing most of the ethanol in the first pipetting step </em></strong><strong></strong></p>
</ul><br/>
  <li>Air  dry the beads on the magnet for 10 minutes. Inspect each tube carefully to  ensure that all of the ethanol has evaporated. It is critical that all residual  ethanol be removed prior to continuing. </li>
  <li>Remove  the tubes from the magnet. </li>
  <li>Add  20 μL Low-EDTA TE Buffer to the dried beads. Mix thoroughly to ensure all beads  are resuspended. </li>
  <li>Transfer  the tubes to the magnet and let stand for 5 minutes to completely clear the  solution of beads. </li>
  <li>Carefully  remove 18 μL of the eluate, ensuring as few beads as possible are carried over  and transfer to a fresh set of PCR tubes.</li>
</ol>
<br/>
<li><strong>Quantitative and Qualitative Assessment of the Library </strong></li><br/>
<ol type="1">
  <li>Quantify  the libraries using a fluorometric assay and/or qPCR. </li>
  <li>Validate  libraries using the hsDNA kit for BioAnalyzer or an equivalent assay. An  example trace using 1 uL of 5 ng/uL library on the hsDNA chip is shown in <strong>Figure  6 </strong>below. </li>
<br/>
<p align="center"><strong>Figure  6. Fragment distribution when 1 μL of 5 ng/μL library is loaded into a High  Sensitivity DNA assay from 200 ng human DNA input.</strong></p><br/>
  <img src="/assets/workflow/Islet Molecular Phenotyping Studies/figure-6.png" width="480" height="280" alt="figure-6-wgbs" style="vertical-align:middle" /><br/>
<br/>
  <li>Normalize and pool  libraries following the Illumina guidelines &ldquo;Best practices for manually nor-  malizing library concentrations&rdquo; and the &ldquo;Low-Diversity Sequencing&rdquo; guidelines  for your specific sequencer. See Appendix A. of this guide for guidelines on  color balancing and multiplexing of Tecan libraries. </li><br/>
</ol> 
  
  <li><strong>Barcode Sequences and Guidelines for Multiplexing </strong></li><br/>
<ol type="1">
  <li>Barcode  sequences and multiplexing guidelines for adaptors used in the Ultralow  Methyl-Seq library prep kit can be found in Tables 12 and 13. </li>
  <li>All  barcode sequences are separated by an edit distance of three. For further  details on the barcode design strategy, please refer to Faircloth BC, Glenn TC  (2012) Not All Sequence Tags Are Created Equal: Designing and Validating  Sequence Identification Tags Robust to Indels. PLoS ONE 7(8): e42543.  doi:10.1371/journal.pone.0042543. </li>
<br/>  
<p align="center"><strong>Table 12. Barcode sequences for adaptors used  in the Ultralow Methyl-Seq 32-plex Adaptor Plate (S02694, Part No. 0366)</strong></p>
<ul><br />
  <p><strong><em>Note:<br/>Barcodes in the Ultralow  Methyl-Seq 32-plex Adaptor Plate are color balanced in pairs (i.e. A01 + B01, C01 + D01, etc.) and in sets of 8 by  column.</em></strong></p></ul>
<br/>

  <table border="1" align="center" cellspacing="0" cellpadding="0">
    <tr>
      <td bgcolor="#9ebddc"><p align="center"><strong>PLATE POSITION </strong></p></td>
      <td bgcolor="#9ebddc"><p align="center"><strong>BARCODE SEQUENCE </strong></p></td>
      <td bgcolor="#9ebddc"><p align="center"><strong>PLATE POSITION </strong></p></td>
      <td bgcolor="#9ebddc"><p align="center"><strong>BARCODE SEQUENCE </strong></p></td>
  </tr>
    <tr>
      <td><p align="center">A01 </p></td>
      <td><p align="center">CGCTACAT </p></td>
      <td><p align="center">A03 </p></td>
      <td><p align="center">GTACACCT </p></td>
    </tr>
    <tr>
      <td><p align="center">B01 </p></td>
      <td><p align="center">AATCCAGC </p></td>
      <td><p align="center">B03 </p></td>
      <td><p align="center">ACGAGAAC </p></td>
    </tr>
    <tr>
      <td><p align="center">C01 </p></td>
      <td><p align="center">CGTCTAAC </p></td>
      <td><p align="center">C03 </p></td>
      <td><p align="center">CGACCTAA </p></td>
    </tr>
    <tr>
      <td><p align="center">D01 </p></td>
      <td><p align="center">AACTCGGA </p></td>
      <td><p align="center">D03 </p></td>
      <td><p align="center">TACATCGG </p></td>
    </tr>
    <tr>
      <td><p align="center">E01 </p></td>
      <td><p align="center">GTCGAGAA </p></td>
      <td><p align="center">E03 </p></td>
      <td><p align="center">ATCGTCTC </p></td>
    </tr>
    <tr>
      <td><p align="center">F01 </p></td>
      <td><p align="center">ACAACAGC </p></td>
      <td><p align="center">F03 </p></td>
      <td><p align="center">CCAACACT </p></td>
    </tr>
    <tr>
      <td><p align="center">G01 </p></td>
      <td><p align="center">ATGACAGG </p></td>
      <td><p align="center">G03 </p></td>
      <td><p align="center">TCTAGGAG </p></td>
    </tr>
    <tr>
      <td><p align="center">H01 </p></td>
      <td><p align="center">GCACACAA </p></td>
      <td><p align="center">H03 </p></td>
      <td><p align="center">CTCGAACA </p></td>
    </tr>
    <tr>
      <td><p align="center">A02 </p></td>
      <td><p align="center">CTCCTAGT </p></td>
      <td><p align="center">A04 </p></td>
      <td><p align="center">ACGGACTT </p></td>
    </tr>
    <tr>
      <td><p align="center">B02 </p></td>
      <td><p align="center">TCTTCGAC </p></td>
      <td><p align="center">B04 </p></td>
      <td><p align="center">CTAAGACC </p></td>
    </tr>
    <tr>
      <td><p align="center">C02 </p></td>
      <td><p align="center">GACTACGA </p></td>
      <td><p align="center">C04 </p></td>
      <td><p align="center">AACCGAAC </p></td>
    </tr>
    <tr>
      <td><p align="center">D02 </p></td>
      <td><p align="center">ACTCCTAC </p></td>
      <td><p align="center">D04 </p></td>
      <td><p align="center">CCTTAGGT </p></td>
    </tr>
    <tr>
      <td><p align="center">E02 </p></td>
      <td><p align="center">CTTCCTTC </p></td>
      <td><p align="center">E04 </p></td>
      <td><p align="center">CCTATACC </p></td>
    </tr>
    <tr>
      <td><p align="center">F02 </p></td>
      <td><p align="center">ACCATCCT </p></td>
      <td><p align="center">F04 </p></td>
      <td><p align="center">AACGCCTT </p></td>
    </tr>
    <tr>
      <td><p align="center">G02 </p></td>
      <td><p align="center">CGTCCATT </p></td>
      <td><p align="center">G04 </p></td>
      <td><p align="center">TCCATTGC </p></td>
    </tr>
    <tr>
      <td><p align="center">H02 </p></td>
      <td><p align="center">AACTTGCC </p></td>
      <td><p align="center">H04 </p></td>
      <td><p align="center">CAAGCCAA </p></td>
    </tr>
  </table>
<br/>
</ol>
  <li><strong>Oxidant Color Changes </strong></li><br/>
<ol type="1">
  <li>Upon  receipt of the kit, the color of the thawed Oxidant Solution should resemble  Solution A in <strong>Figure7</strong>. </li>
  <li>If  the Oxidant Solution looks dark yellow or green with considerable amounts of  black precipitate (Solution D or E), it is a sign that the reagent has become  exposed to contaminants or CO2. If you suspect that this has occurred, please  do not use the Oxidant Solution and contact Tecan NGS technical support for  advice. </li>
<br/>
<p align="center"><strong>Figure 7. Expected Oxidant Solution Color  Changes</strong></p><br/>
  <img src="/assets/workflow/Islet Molecular Phenotyping Studies/figure-7.png" width="390" height="290" alt="oxidation-color-change" style="vertical-align:middle" /></p>
<br/>
  <li>Figure  description-</li><br/>
<ol type="A">
  <b><li>Oxidant Solution stock  concentration supplied in the TrueMethyl oxBS Module. </li></b>
  <b><li>10-fold dilution of  the Oxidant Solution in alkaline solution. </li></b>
  <b><li>Working oxidant  concentration (WOC). </li></b>
  <b><li>5:1 molar excess of  WOC:ethanol. </li></b>
 <b><li>E. 20:1 molar excess  of ethanol:WOC. </li></b><br/>
</ol>

  <li>During  oxidation (section V. I.), the color of the oxidation reaction should be  similar to Solution B or C in <strong>Figure 7</strong>. It is normal for a small amount  of black precipitate to form during the oxidation reaction. The purpose of the  strong centrifugation step following oxidation is to pellet the dark  precipitate and enable removal of the clear orange/yellow solution without this  contaminant. After the oxidation reaction and subsequent 10 minute  centrifugation are complete, the solution should remain orange indicating a  suc- cessful oxidation. </li>
  <li>If  the post-oxidation color appears as Solution D, it suggests partial decomposition  of the oxidant, indi- cating that oxidation of the DNA samples was successful.  However, if the post-oxidation color resembles Solution E, significant  decomposition of the oxidant has likely occurred, resulting in incomplete con-  version of 5-hmC --&gt; U. In this instance, it is recommended to re-purify the  sample in order to remove contaminants from the starting DNA sample solution.  To avoid contaminants, ensure all guidance regard- ing oxidation solution  sensitivity in section III. D. is followed, including the use of only Ultra  Pure Water provided with the TrueMethyl oxBS Module in steps containing the  oxidant. </li>
  <li>As  a final note on color changes, samples that have been processed using the  Oxidant Solution may also take on a light grey color after addition of the  Bisulfite Conversion Solution (section V. J.). This is normal and will not  impact downstream processing. </li>
<ol type="a">
 <li>Next  steps-</li>
  <li>Qubit and BioAnalyzer to do the concentration </li>
  <li>Kapa concentration using Kapa kit (KK4808) </li>
  <li>Me-seq for balance </li>
  <li>Nova-seq 6000 for final sequencing </li>
</ol>
</ol>
</ol>


</body>
</html>
